<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">eLife</span> <span class="hwp" title="hwp">eLife</span> <span class="publisher-id" title="publisher-id">eLife</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">eLife</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">2050-084X</span> <span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4606454">4606454</a> </span> <span class="publisher-id" title="publisher-id">06671</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.06671">10.7554/eLife.06671</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Genes and Chromosomes</span>: <span class="subject" title="subject">Plant Biology</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Geminivirus-encoded TrAP suppressor inhibits the histone methyltransferase SUVH4/KYP to counter host defense </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castillo-González</span> <span class="given-names" tagx="given-names" title="given-names">Claudia</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#con1" /> <a href="#conf1" /> <a href="#dataro1" /> <a href="#dataro2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Xiuying</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#aff3">3</a> <a href="#con3" /> <a href="#conf1" /> <a href="#dataro1" /> <a href="#dataro2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">Changjun</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#aff4">4</a> <a href="#aff5">5</a> <a href="#con5" /> <a href="#conf1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Changjiang</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#aff6">6</a> <a href="#con6" /> <a href="#conf1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Zeyang</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#con11" /> <a href="#conf1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">Tao</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#aff4">4</a> <a href="#con7" /> <a href="#conf1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Feng</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#aff7">7</a> <a href="#con8" /> <a href="#conf1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Yijun</span> </span> <a href="#aff7">7</a> <a href="#con9" /> <a href="#conf1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Xueping</span> </span> <a href="#aff8">8</a> <a href="#con10" /> <a href="#conf1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Xiu-Jie</span> </span> <a href="#aff3">3</a> <a href="#con4" /> <a href="#conf1" /> <a href="#dataro1" /> <a href="#dataro2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Xiuren</span> </span> <a href="#aff1">1</a> <a href="#aff2">2</a> <a href="#cor1">*</a> <a href="#par-1" /> <a href="#par-2" /> <a href="#par-3" /> <a href="#con2" /> <a href="#conf1" /> <a href="#dataro1" /> <a href="#dataro2" /> </span> <span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Department of Biochemistry and Biophysics</span> <span class="institution" title="institution">Texas A&amp;amp;M University</span> <span class="addr-line" title="addr-line">College Station</span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Institute for Plant Genomics and Biotechnology</span> <span class="institution" title="institution">Texas A&amp;amp;M University</span> <span class="addr-line" title="addr-line">College Station</span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">State Key Laboratory of Plant Genomics, Institute of Genetics and Developmental Biology</span> <span class="institution" title="institution">Chinese Academy of Sciences</span> <span class="addr-line" title="addr-line">Beijing</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">State Key Laboratory of Rice Biology, Institute of Biotechnology</span> <span class="institution" title="institution">Zhejiang University</span> <span class="addr-line" title="addr-line">Hangzhou</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff5">[5], <span class="institution" title="institution">Yunnan Academy of Tobacco Agricultural Sciences</span> <span class="addr-line" title="addr-line">Yunnan</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff6">[6], <span class="institution" title="institution">College of Agronomy</span> <span class="institution" title="institution">Heilongjiang Bayi Agricultural University</span> <span class="addr-line" title="addr-line">Daqing</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff7">[7], <span class="institution" title="institution">Institute of Plant Protection</span> <span class="institution" title="institution">Jiangsu Academy of Agricultural Sciences</span> <span class="addr-line" title="addr-line">Nanjing</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="aff8">[8], <span class="institution" title="institution">Biotechnology Institute, College of Agriculture &amp;amp; Biotechnology</span> <span class="institution" title="institution">Zhejiang University</span> <span class="addr-line" title="addr-line">Zhejiang</span> <span class="country" tagx="country" title="country">China</span> </span> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">Detlef</span> </span> <span class="role" tagx="role" title="role">Reviewing editor</span> <span class="citation_author_institution" id="">[], <span class="institution" title="institution">Max Planck Institute for Developmental Biology</span> <span class="country" tagx="country" title="country">Germany</span> </span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>For correspondence: <span class="email" tagx="email" title="email">xiuren.zhang@tamu.edu</span> </div> </div> <span class="pub-date-" title="pub-date-">: <span>2015-9-9</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2015</span> </span> <span class="volume" tagx="volume" title="volume">4</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e06671</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-1-25</span> <span class="accepted" title="accepted">accepted: 2015-9-05</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Castillo-González et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2015</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited. </span> </span> </div> <a class="self-uri" href="elife06671.pdf" title="self-uri">elife06671.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Transcriptional gene silencing (TGS) can serve as an innate immunity against invading DNA viruses throughout Eukaryotes. Geminivirus code for TrAP protein to suppress the TGS pathway. Here, we identified an <i>Arabidopsis</i> H3K9me2 histone methyltransferase, Su(var)3-9 homolog 4/Kryptonite (SUVH4/KYP), as a bona fide cellular target of TrAP. TrAP interacts with the catalytic domain of KYP and inhibits its activity in vitro. TrAP elicits developmental anomalies phenocopying several TGS mutants, reduces the repressive H3K9me2 mark and CHH DNA methylation, and reactivates numerous endogenous KYP-repressed loci in vivo. Moreover, KYP binds to the viral chromatin and controls its methylation to combat virus infection. Notably, <i>kyp</i> mutants support systemic infection of TrAP-deficient Geminivirus. We conclude that TrAP attenuates the TGS of the viral chromatin by inhibiting KYP activity to evade host surveillance. These findings provide new insight on the molecular arms race between host antiviral defense and virus counter defense at an epigenetic level. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.001">http://dx.doi.org/10.7554/eLife.06671.001</a> </p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
eLife digest</div> <p>Many viruses can infect plants and cause diseases that can reduce crop yields. The Geminiviruses are a family of plant viruses that are transmitted by insects and infect tomato, cabbage, and many other crop plants. These viruses hijack the plant cells that they infect and force the plant cells to make viral proteins using instructions provided by the genes in the virus' own DNA. To make proteins, DNA is first copied into molecules of messenger ribonucleic acid (or mRNA) in a process called transcription. However, plants can defend themselves by blocking the transcription of viral DNA through ‘transcriptional gene silencing’. </p> <p>In plant cells, DNA is packaged around proteins called histones to form a structure called chromatin. Small chemical tags attached to the histones can alter the structure of chromatin to regulate the activity of the genes encoded within it. For example, ‘methyl’ tags added to certain histones can block transcription and lower the activity of a gene. DNA from viruses can also associate with histones inside plant cells meaning that transcriptional gene silencing can take place by the addition of these methyl tags. Many Geminiviruses produce a protein called TrAP, which can activate transcription, but it is not clear how this works. </p> <p>Castillo-González et al. studied the TrAP proteins from two different Geminiviruses that can infect crop plants. The commonly used model plant, <i>Arabidopsis thaliana</i>, was genetically engineered to produce high levels of these TrAP proteins. These ‘transgenic’ plants did not develop properly: they grew more slowly, had abnormal leaves, and flowered earlier. Furthermore, hundreds of plant genes were more active than usual in the transgenic plants, which suggests that TrAP inhibits transcriptional gene silencing. Further experiments showed that TrAP directly binds to a plant enzyme called KYP—which normally deposits methyl groups on chromatin and prevents it from working. This reduces the number of methyl groups that are attached to histones associated with both viral and plant chromatin, which results in the activation of genes that would normally be switched off. </p> <p>Castillo-González et al.'s findings show how Geminiviruses can stop transcriptional gene silencing of chromatin that contains virus DNA to evade the host plant's defenses. The next challenge is to understand how TrAP inhibits KYP, which may present new ways to genetically engineer plants to become resistant to infection by viruses. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.002">http://dx.doi.org/10.7554/eLife.06671.002</a> </p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Author keywords</div> <div class="kwd" title="kwd">
epigenetic silencing</div> <div class="kwd" title="kwd">
viral suppression</div> <div class="kwd" title="kwd">
geminivirus</div> <div class="kwd" title="kwd">
histone methyltransferase</div> <div class="kwd" title="kwd">
TrAP/AL2/AC2</div> <div class="kwd" title="kwd">
host–virus interaction</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd"> <i>Arabidopsis</i> </div> <div class="kwd" title="kwd">
viruses</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span> <span class="institution" title="institution">National Institutes of Health (NIH)</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">R21 AI097570</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Xiuren</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000928</span> <span class="institution" title="institution">Welch Foundation</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">A-1777</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Xiuren</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span> <span class="institution" title="institution">National Science Foundation (NSF)</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">MCB-1253369</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Xiuren</span> </span> </span> </div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication. </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="elife-xml-version" value="2.3" /> <meta name="Author impact statement" value="A viral protein hijacks the key effector of transcriptional gene silencing, Su(var)3-9, to evade host plant surveillance." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>RNA silencing is a host defense mechanism to combat invading nucleic acids. One type of RNA silencing is referred to as post-transcriptional gene silencing (PTGS). In PTGS, double-stranded RNAs (dsRNAs) are processed by Dicer-like ribonucleases into small-interfering RNAs (siRNAs). Mature siRNAs are incorporated into an Argonaute (AGO)-centered RNA-induced silencing complex (RISC) to regulate expression of target genes through RNA cleavage or translational repression. PTGS has evolved as a universal defense response toward all viruses because dsRNAs can result from intermediates in RNA virus replication, highly structured RNA virus genomes, or from viral transcripts. To evade this surveillance mechanism, virtually all plant viruses are known to encode suppressor proteins that are able to block different key steps of the PTGS pathway ( <a href="#bib12">Ding and Voinnet, 2007</a>). </p> <p>While the host/virus battle at the PTGS level has been well appreciated, virus suppression at a transcriptional gene silencing (TGS) level is poorly understood. In eukaryotes, the nuclear DNA is wrapped onto histone octamers to constitute chromatin. The chromatin undergoes various DNA and histone methylations, and these modifications have variable effects on gene expression depending on the precise residues, contexts, and modification complexity. Histone methylation takes place on lysine and arginine residues of the amino-terminal tails ( <a href="#bib28">Kouzarides, 2007</a>; <a href="#bib23">Greer and Shi, 2012</a>). The prevailing dogma is that histone 3 lysine 4 tri-methylation (H3K4me3) is mostly associated with transcriptionally active euchromatin, while H3K9me2 and H3K27me3 are repressive marks ( <a href="#bib11">Deal and Henikoff, 2011</a>; <a href="#bib20">Feng and Jacobsen, 2011</a>). Histone methylation is catalyzed by SET domain containing methyltransferases, specifically, H3K9me2 is deposited by <i>Arabidopsis</i> Su(var)3-9 homolog 4, Kryptonite (KYP) ( <a href="#bib14">Du et al., 2014a</a>), and its paralogs (SUVH5,6), while H3K27 methylation is carried out by the Polycomb repressive complex 2 (PRC2) <i>,</i> which includes Curly Leaf (CLF) ( <a href="#bib31">Liu et al., 2010</a>; <a href="#bib66">Zheng and Chen, 2011</a>). Local H3K9me2 and H3K27me3 can spread over wide regions to elicit heterochromatin configuration. In animals, the propagation of histone methylation entails co-repressor heterochromatin protein 1 (HP1), whereas in plants, KYP acts synergistically with DNA methyltransferases (i.e., Chromomethylase 3 [CMT3]) to constitute a mutually reinforcing cycle of DNA and histone methylation to secure TGS ( <a href="#bib16">Du et al., 2012</a>, <a href="#bib14">2014a</a>). </p> <p>Histone methylation not only regulates endogenous gene expression but also invasive DNAs such as transposons and viruses ( <a href="#bib38">Narasipura et al., 2014</a>). Plant DNA viruses, exemplified by Geminivirus, form minichromosomes in the host ( <a href="#bib24">Hanley-Bowdoin et al., 2013</a>). Both Geminivirus DNA and associated histones are methylated in infected cells, whereas viral methylation is reduced in methylation-deficient hosts, methylation-compromised <i>Arabidopsis</i> mutants are hypersusceptiple to Geminivirus infection and show exacerbated disease symptoms ( <a href="#bib42">Raja et al., 2008</a>). Thus, plants appear to employ methylation of viral chromatin to limit viral replication and transcription ( <a href="#bib1">Aregger et al., 2012</a>; <a href="#bib40">Pumplin and Voinnet, 2013</a>). On the other hand, Geminiviruses encode a multi-functional protein called transcriptional activation protein (TrAP/AL2/AC2) that counters the epigenetic defense ( <a href="#bib42">Raja et al., 2008</a>; <a href="#bib6">Buchmann et al., 2009</a>). It has been shown that TrAP inhibits adenosine kinase (ADK) ( <a href="#bib54">Wang et al., 2005</a>). ADK catalyzes the synthesis of 5′ AMP from adenosine and ATP, a process that promotes the regeneration of S-adenosyl-methionine (SAM), the major methyl donor in the cell ( <a href="#bib34">Moffatt et al., 2002</a>; <a href="#bib6">Buchmann et al., 2009</a>). Consequently, the TrAP-mediated inhibition of ADK activity likely impedes downstream trans-methylation events, including viral chromatin methylation in the nucleus ( <a href="#bib4">Bisaro, 2006</a>; <a href="#bib6">Buchmann et al., 2009</a>). In parallel, some Geminivirus encode a TrAP positional homolog, named C2, that is able to stabilize SAM decarboxylase 1 to downregulate the methyl group metabolism ( <a href="#bib65">Zhang et al., 2011</a>). It seems that interfering with the methyl cycle is a common suppression mechanism for Geminivirus-encoded TrAP/AL2/C2 proteins. In addition, C2 also subverts the activity of COP9 signalosome to inhibit jasmonate signaling ( <a href="#bib33">Lozano-Durán et al., 2011</a>), suggesting its multiple functions in viral counter-defense. </p> <p>Here, we investigated the suppression mechanism of TrAP proteins, encoded by two Geminivirus members, <i>Tomato Golden Mosaic Virus</i> (TGMV) and <i>Cabbage Leaf Curl Virus</i> (CaLCuV). We found that constitutive expression of TGMV- <i>TrAP</i> in <i>Arabidopsis thaliana</i> caused morphological abnormalities that mimic loss-of-function mutants of numerous TGS components including <i>lhp1</i> ( <i>like-heterochromatin1</i>) and <i>clf</i>. Microarray analyses of <i>TrAP</i> transgenic plants and <i>lhp1</i> mutants revealed a substantial overlap in reprogrammed host genes at a genome-wide level. Through biochemical screening, we identified KYP as the bona fide target of TrAP. We demonstrated in vitro that TrAP binds to the catalytic domain of KYP and inhibits its enzymatic activity; while in vivo, <i>TrAP</i> decreases the repressive H3K9me2 marks and H3K9me2-dependent CHH methylation in gene-rich regions. We also found that KYP directly associates with the Geminivirus minichromosome and deposits H3K9me2 marks on viral chromatin. In addition, <i>kyp</i> mutants but not wild-type plants sustain low systemic infection of CaLCuV lacking TrAP protein. Taken together, we propose that KYP-catalyzed H3K9me2 is a line of the innate immunity against invading DNA pathogens, and Geminivirus TrAP functions to inactivate KYP to counter host defense. Thus, this study provides new insight into the host–virus interaction at the TGS level. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="tgmv-encodedtrapcausesdevelopmentalabnormalitiesinarabidopsisbutnotthroughmirnapathway" title="sec"> <div class="title" tagx="title" title="title">
TGMV-encoded TrAP causes developmental abnormalities in <i>Arabidopsis</i> but not through miRNA pathway </div> <p>To study the suppression mechanism of TrAP, we generated 235 <i>Arabidopsis</i> transgenic lines overexpressing full-length TGMV TrAP, with or without Flag-Myc4, 3HA, or CFP epitopes. These transgenic plants were confirmed by northern (data not shown) or western blot assays ( <a href="#fig1">Figure 1A</a> and <a href="#fig3s1">Figure 3—figure supplement 1A</a>). Importantly, the majority of the transgenic lines exhibited developmental abnormalities consisting of short statues, strongly upward curled cotyledons and true leaves ( <a href="#fig1">Figure 1B</a>). Moreover, these overexpressing lines exhibited early flowering compared to wild-type (WT) plants. These phenotypes were morphologically distinct from loss-of-function mutants of <i>ADK1 ADK2</i> ( <a href="#bib56">Weretilnyk et al., 2001</a>; <a href="#bib34">Moffatt et al., 2002</a>), <i>SnRK1</i> ( <a href="#bib46">Shen et al., 2009</a>, <a href="#bib45">2014</a>), <i>PEAPOD2</i> ( <a href="#bib30">Lacatus and Sunter, 2009</a>), and <i>rgsCaM</i> ( <a href="#bib8">Chung et al., 2014</a>), a calmodulin-like protein, which are also targets or partners of TrAP. This result indicated that TrAP exerts some novel cellular function (s). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.003"> <span class="label" tagx="label" title="label">Figure 1.</span> </a> </div>   <div class="title" tagx="title" title="title">
TrAP caused developmental abnormalities in Arabidopsis but not through miRNA pathway.</div> <p>( <b>A</b>) Western blot analysis of <i>35S-TrAP-3HA</i> and <i>35S-TrAP-CFP</i> transgenic lines. Arrows indicate the locations of the tagged TrAP proteins; * Cross-reaction band serves a loading control. ( <b>B</b>) Morphological defects of Arabidopsis transgenic plants expressing <i>35S-TGMV TrAP</i>. Photographs were taken of 10-day seedlings. ( <b>C</b>) RNA blot analysis of <i>PHB</i> and <i>AGO1</i> transcripts in the <i>TrAP</i> overexpression transgenic plants using gene-specific random-labeled probes. <i>25S rRNA</i> is a loading control. ( <b>D</b>) sRNA blot analysis of miRNA and miRNA* in the <i>TrAP</i> overexpression transgenic plants. Total RNA was prepared from a pool of T2 transgenic plants (n &amp;gt; 50 for each line). sRNA blots were probed using 5′ end <sup>32</sup>P-labeled oligonucleotide probes complementary to the indicated miRNA or miRNA*. <i>5S rRNA</i> and <i>tRNA</i> are a loading control. All the samples were run in the same gel; but the lane order of miRNA*s was rearranged to match that of miRNAs. ( <b>E</b>) sRNA blot analysis of miR165 loading into Arabidopsis RNA-induced silencing complexes (RISCs). RNA was extracted from flowers or Flag-AGO1 immunoprecipitates of transgenic plants harboring <i>35S-TrAP</i> or <i>35S-TrAP-3HA</i> in <i>ago1-36; P</i> <sub> <i>AGO1</i> </sub> <i>-Flag-AGO1</i> background and <i>ago1-36; P</i> <sub> <i>AGO1</i> </sub> <i>-Flag-AGO1</i> control plants ( <a href="#bib3">Baumberger and Baulcombe, 2005</a>). Top panel, total RNA; middle panel, each lane contained sRNA associated with Flag-AGO1 immunoprecipitated from 0.4 g of flowers. Bottom panel, the input and immunoprecipitate of Flag-AGO1 were analyzed by Western blot assays in the same samples for sRNA blots. A cross-reacting band (*) was used as a loading control. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.003">http://dx.doi.org/10.7554/eLife.06671.003</a> </p>   </div> <p /> <p>Developmental anomalies of transgenic plants expressing viral suppressors have been associated with interruption of the miRNA pathway. To test this, we compared expression levels of several miRNAs and their targets between Col-0 wild-type and <i>TGMV TrAP</i> transgenic plants. Plants expressing <i>Cucumber mosaic virus</i>-encoded 2b suppressor (Zhang et al., 2006) and <i>ago1-27</i>, a hypomorphic allele of <i>ago1</i> ( <a href="#bib35">Morel et al., 2002</a>), were used as controls. We observed that the accumulation of miR165 and miR168 and their targets, <i>PHB</i> and <i>AGO1</i> transcripts, in the <i>35S-TGMV TrAP</i> lines was comparable to the amount in wild-type plants ( <a href="#fig1">Figure 1C,D</a>). We further confirmed that loading of miRNAs into AGO1-centered RISCs was not affected by TrAP ( <a href="#fig1">Figure 1E</a>). The same results were obtained with miR167, miR159, and ta-siRNA480/255 and their corresponding targets ( <a href="#fig1s1">Figure 1—figure supplement 1</a>). Thus, unlike most of previously reported viral suppressors, TrAP does not act on the miRNA pathway. </p> </div> <div class="trapgeneticallyinterruptsthetgspathway" title="sec"> <div class="title" tagx="title" title="title">
TrAP genetically interrupts the TGS pathway</div> <p>To study how <i>TrAP</i> altered plant development, we mined publicly available databases and literature for the molecular and morphological phenotype of <i>35S-TrAP</i> lines. We found that <i>35S-TrAP</i> transgenic lines phenocopied several mutants in the epigenetic pathway including <i>lhp1</i> ( <a href="#bib27">Kotake et al., 2003</a>) ( <a href="#fig2">Figure 2A</a>) and <i>clf</i> mutants ( <a href="#bib7">Chanvivattana et al., 2004</a>), with respect to the early flowering and upward curling of leaves. CLF belongs to PRC2, a complex that catalyzes the deposition of H3K27me3 marks. LHP1 ( <a href="#bib37">Nakahigashi et al., 2005</a>), on the other hand, associates to silent genes in euchromatin and directs the spreading of the silent status to adjacent loci ( <a href="#bib53">Turck et al., 2007</a>; <a href="#bib60">Zhang et al., 2007a</a>, <a href="#bib62">2007b</a>; <a href="#bib19">Farrona et al., 2008</a>; <a href="#bib66">Zheng and Chen, 2011</a>). Thus, coordinate activities of CLF and LHP1 result in chromatin methylation and transcriptional repression ( <a href="#bib19">Farrona et al., 2008</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.005"> <span class="label" tagx="label" title="label">Figure 2.</span> </a> </div>   <div class="title" tagx="title" title="title">
TrAP is genetically involved in the TGS pathway.</div> <p>( <b>A</b>) <i>35S-TrAP</i> transgenic plants phenocopied <i>lhp1</i> mutants. Photographs were taken of 15-day seedlings. ( <b>B</b>, <b>C</b>) Microarray results were validated by qRT-PCR analysis. Only 12 randomly selected loci were shown. ( <b>D</b>) Gene ontology analysis of the <i>TrAP-</i>regulated DEGs. The numbers adjacent to the pies represent the ratio of genes in each category over the total DEGs. ( <b>E</b>) Genome-wide overlapping of the genes regulated by <i>TrAP</i> and loss-of-function <i>lhp1</i>. White and black numbers correspond to upregulated and downregulated genes, respectively. Maroon number indicates the genes that are differentially deregulated in both genotypes. ( <b>F</b>) Heatmap of the commonly deregulated genes in the <i>35S-TrAP</i> and <i>lhp1</i> lines. The typical gene-ontology categories are shown on top. ( <b>G</b>, <b>H</b>) Microarray and RNA blot analyses of epigenetically regulated flowering genes in the <i>TrAP</i> transgenic lines and <i>lhp1</i> mutants. <i>Cyt450</i> is a control. ( <b>I</b>) qRT-PCR analysis of TEs in heterochromatic regions in the <i>lhp1</i> mutant and <i>TrAP</i> transgenic lines. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.005">http://dx.doi.org/10.7554/eLife.06671.005</a> </p>   </div> <p /> <p>We examined global expression profiles of 7-day-old <i>35S-TrAP</i> transgenic plants compared to Col-0 wild-type using an Affymetrix ATH1 GeneChip and identified 586 genes that are differentially expressed in the <i>35S-TrAP</i> transgenic plants (q &amp;lt; 0.005). Of these, 295 transcripts were elevated whereas 291 were reduced ( <a href="#fig2">Figure 2E</a>). We performed real-time PCR and RNA blot assays to validate the microarray results for the differentially expressed genes (DEGs). Among 25 genes randomly tested, we confirmed the ATH1 results for 24, indicating that the microarray results were reliable ( <a href="#fig2">Figure 2B,C</a>; data not shown). Gene-Ontology (GO) analysis placed the DEGs into seven functional categories ( <a href="#fig2">Figure 2D</a>; <a href="#SD1-data">Supplementary file 1</a>): hormone response (86 genes), stress response (94 genes), development regulation (50 genes), transcriptional regulation (20 genes), RNA metabolism (13 genes), post-translational modification (36 genes), and general metabolism (182 genes), plus a set of 105 un-annotated genes ( <a href="#fig2">Figure 2D</a>; <a href="#SD1-data">Supplementary file 1</a>). </p> <p>Then, we compared the DEG profiles of the <i>TrAP</i> overexpression lines and loss-of-function <i>lhp1</i> mutant ( <a href="#fig2">Figure 2E,F</a>; <a href="#SD2-data">Supplementary file 2</a>). Transcriptome analysis revealed that out of 295 genes significantly upregulated in the <i>35S-TrAP</i> transgenic lines, 120 (40.7%) were also upregulated in <i>lhp1</i> mutant. This is significantly greater than 1.28% expected by chance (p &amp;lt; 2.2e <sup>−16</sup>, Pearson's Chi-squared test). Interestingly, among the co-upregulated genes are a group of flowering-stimulated transcriptional factors including the key flowering-time integrator, <i>FT,</i> and 12 other genes such as <i>TFL1</i>, <i>AGL5</i>, and <i>AGL9</i> ( <a href="#bib19">Farrona et al., 2008</a>) ( <a href="#fig2">Figure 2F–H</a>; <a href="#SD2-data">Supplementary file 2</a>; <a href="#SD8-data">Supplementary file 8</a>) clustered in the transcriptional regulation category. Importantly, all these genes are regulated through epigenetic pathways and account for the early flowering phenotypes of <i>lhp1</i> mutant and possibly of <i>TrAP</i> transgenic plants as well ( <a href="#bib22">Gan et al., 2013</a>). Other highly represented categories included 31 genes involved in aging and 116 genes engaged in stress responses ( <a href="#SD2-data">Supplementary file 2</a>). Notably, the stress-responsive genes included genes specific to biotic stress such as <i>PR4</i>, <i>WRKY18</i>, <i>FLS2</i>, and <i>PDF1.2</i>; additionally, genes related to chemical stress, such as <i>PTR3</i> and <i>TAT3</i>, were also identified. Thus, constitutive expression of <i>TrAP</i> could trigger plant senescence and innate defense pathways, and this activation is potentially through interference with the <i>LHP1</i>-related epigenetic silencing. </p> <p>Similarly, out of 291 genes significantly downregulated in the <i>35S-TrAP</i> transgenic lines, 137 (47.1%) were also repressed in <i>lhp1</i> mutant. This is significantly greater than 1.25% expected by chance (p &amp;lt; 2.2e <sup>−16</sup>, Pearson's Chi-squared test) ( <a href="#SD2-data">Supplementary file 2</a>). Genes related to auxin response were of special interest. Of the 32 DEGs involved in the auxin pathway, 29 genes were downregulated in the <i>35S-TrAP</i> transgenic plants, classified as small auxin upregulated mRNAs ( <i>SAUR</i>s). These results suggested a possible hyposensitivity to auxin in <i>lhp1</i> mutants and <i>TrAP</i> transgenic plants, which could explain the smaller statues of both genotypes. Consistent with this hypothesis, numerous auxin-repressed loci including <i>PS2</i> and aging genes like <i>TET9</i>, <i>SAG13</i>, and <i>SRG1</i> were upregulated in both lines. Concomitantly, six genes related to cell growth and five genes engaged in cell wall loosening were also repressed. Significant genome-wide overlap of <i>35S-TrAP</i> and <i>lhp1</i> loss-of-function-responsive genes suggested that <i>TrAP</i> might be genetically involved in the <i>LHP1</i>-related TGS pathway. </p> <p>Since LHP1 is believed to reside in euchromatic regions, we wondered whether <i>TrAP</i> also deregulates expression of heterochromatic loci. To this end, we selected numerous transposable elements (TEs) that were not recovered from the microarray assays and assessed them directly by qRT-PCR. Excitingly, most of the tested transposons were transcriptionally active ( <a href="#fig2">Figure 2I</a>), further suggesting that <i>TrAP</i> indeed inhibits the TGS pathway, in both euchromatic and heterochromatic regions. </p> </div> <div class="kypisabonafidetargetoftrap" title="sec"> <div class="title" tagx="title" title="title">
KYP is a bona fide target of TrAP</div> <p>Given that <i>TrAP</i> transgenic plants phenocopied several TGS mutants and displayed transcriptional activation of heterochromatic loci, we hypothesized that TrAP epistatically regulates a TGS integrator (s), indirectly leading to deregulation of the epigenetic marks. Analysis of the microarray data challenged this possibility as no significant changes in the transcripts of any canonical TGS components were revealed ( <a href="#SD3-data">Supplementary file 3</a>). An alternative hypothesis was that TrAP directly interferes with the function of a TGS component (s). To test this in an unbiased manner, we used luciferase complementation imaging (LCI) assay to screen 34 TGS-related proteins and some other cellular factors for TrAP interaction ( <a href="#bib65">Zhang et al., 2011</a>) ( <a href="#SD4-data">Supplementary file 4</a>). In the LCI experiments, the N- and C-terminal parts of firefly luciferase (NLuc and CLuc) are fused to different test proteins to be transiently expressed in <i>Nicotiana benthamiana</i> ( <i>N. benthamiana</i>). When NLuc and CLuc are brought together through interaction of the test proteins, catalytic activity is restored and recorded through CCD camera. In our LCI screening, we recovered LHP1 and KYP, a SUVH-type H3K9me2 methyltransferase, suggesting that TrAP is physically close to LHP1 and/or KYP proteins in vivo ( <a href="#fig3">Figure 3B</a>). Next, we carried out confocal microscopy imaging assays. TrAP co-localized with both LHP1 and KYP in scattered but not yet clearly defined nuclear foci, whereas co-expression of TrAP-CFP and other YFP-tagged proteins in <i>N. benthamiana</i> cells did not show such patterns ( <a href="#fig3">Figure 3C</a>). These observations further suggested that TrAP was in the same complexes as LHP1 or KYP. To further examine if TrAP interacted with these proteins, we conducted co-immunoprecipitation (Co-IP) assays ( <a href="#fig3">Figure 3D</a>). Interestingly, we validated the TrAP-KYP interaction ( <a href="#fig3">Figure 3D</a>) but not TrAP-LHP1 (data not shown), indicating that the LCI signal resulting from the TrAP-LHP1 combination likely involved additional cofactors between TrAP and LHP1 in vivo ( <a href="#fig3">Figure 3B</a>). Alternatively, TrAP-LHP1 interaction might be transient or unstable in our stringent co-IP condition. We observed that expression of CLuc-HA3-KYP in <i>N. benthamiana</i> yielded truncated proteins of various lengths that accumulated to comparable levels as the full-length protein; only the full-length KYP showed specific interaction with TrAP, implying the KYP C-terminal domain as the interaction interface with TrAP ( <a href="#fig3">Figure 3D</a>). We further confirmed the in vivo TrAP-KYP interaction by Förster resonance energy transfer-acceptor bleaching (FRET-AB), using TrAP-CFP as a donor and YFP-KYP as an acceptor. Shortly, FRET involves the energy transfer from an excited donor to an adjacent acceptor when the fluorophores are less than 10 nm apart. If the fluorophores are coupled, the excited donor leads to acceptor emission; bleaching the acceptor allows the emission of the donor to be measured ( <a href="#fig3">Figure 3E</a>). Consecutive cycles of YFP-KYP bleach/recovery correlated with release/quench on the TrAP-CFP signal ( <a href="#fig3">Figure 3E,F</a>); the positive FRET-AB (CaLCuV-TrAP 2.699 ± 0.23, TGMV-TrAP 3.8228 ± 0.58, p &amp;lt; 0.05) corroborated TrAP-CFP/YFP-KYP interaction ( <a href="#fig3">Figure 3G</a>). To examine whether TrAP interacted with KYP under physiological conditions, we isolated TrAP complexes through two-step immunoprecipitation ( <a href="#bib67">Zhu et al., 2013</a>) from stable transgenic plants expressing <i>Flag-Myc4-TrAP</i> under the inducible promoter ( <i>XVE</i>) ( <a href="#bib68">Zuo et al., 2000</a>) followed by mass spectrometry analysis. A total of 624 peptides representing 288 unique sequences were recovered from the TrAP sample; of those, 31 unique peptides matched specifically to KYP/SUVH4 and were not found in control immunoprecipitates using Col-0 plants ( <a href="#fig3s1">Figure 3—figure supplement 1</a>). Together, all these assays clearly indicated that TrAP and KYP interact in vivo. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.006"> <span class="label" tagx="label" title="label">Figure 3.</span> </a> </div>   <div class="title" tagx="title" title="title">
TrAP interacts with KYP in vivo.</div> <p>( <b>A</b>) Schematic representation of the luciferase complementation imaging assay shows the different combinations of infiltrated constructs that were fused either to N-terminal (NLuc) and C-terminal (CLuc) regions of luciferase. ( <b>B</b>) Screening of host factors targeted by TrAP. The infiltration positions of the constructs (red arrows) and luminescence signal resulting from the protein–protein interaction in a leaf are shown. FTSZ and LOM2 serve as negative controls. ( <b>C</b>) Confocal imaging assays show the co-localization of TrAP-CFP with YFP-KYP and YFP-LHP1 in the nuclei in <i>Nicotiana benthamiana</i>. FTSZ serves as a negative control. ( <b>D</b>) Specific interaction between KYP and TrAP was confirmed in <i>N. benthamiana</i> by co-immunoprecipitation (Co-IP). Constructs harboring <i>35S-Myc-TrAP-nLuc</i> and <i>cLuc-HA-KYP</i> were co-infiltrated in <i>N. benthamiana</i> leaves. IP was conducted using anti-Myc antibody. Western blot analyses were done with the crude extract (input) and the IP products using anti-Myc or -HA antibodies. Truncated versions (red arrows) serve as an internal control. ( <b>E</b>) Exemplary imaging of FRET assays of TrAP-CFP and YFP-KYP co-expressed in a nucleus. The nucleus is irradiated with 458 nm laser to excite the CFP fluorophore. Three regions were selected for the assay: #1, autofluorescence control, #2 fluorophore decay control, and #3, FRET-acceptor bleaching test. Regions #1 and #3 were treated with pulses of 514 nm laser to bleach the YFP fluorophore. The CFP signal is then visible in region #3 when the emission of CFP is dequenched. ( <b>F</b>) Quantification of the signals from each fluorophore observed during FRET-AB experiment in <b>E</b>. ( <b>G</b>) FRET is positive for YFP-KYP paired with the CFP-tagged TrAPs from either CaLCuV or TGMV. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.006">http://dx.doi.org/10.7554/eLife.06671.006</a> </p>   </div> <p /> </div> <div class="trapbindstothecatalyticdomainofkyp" title="sec"> <div class="title" tagx="title" title="title">
TrAP binds to the catalytic domain of KYP</div> <p>We performed in vitro pull-down assays to examine whether TrAP interacts directly with KYP. We found that maltose-binding protein (MBP)-KYP, but not MBP and other MBP-tagged control proteins, was able to pull down GST-tagged TGMV- and CaLCuV-encoded TrAP proteins ( <a href="#fig4">Figure 4A,B</a>). This interaction was specific as MBP-KYP was unable to pull down GST protein alone. Notably, we did not observe direct LHP1-TrAP interaction in the parallel pull-down experiments ( <a href="#fig4s1">Figure 4—figure supplement 1</a>); and this result was consistent with the in vivo Co-IP experiments. From N- to C- termini, KYP contains SRA, PreSET, SET, and PostSET domains. To further define the protein domain (s) responsible for the specific interaction, we generated five truncations of KYP ( <a href="#fig4">Figure 4C</a>). Pull-down assays showed that the SET domain interacted with TrAP proteins from both TGMV and CaLCuV at a higher affinity, whereas SRA domain might interact with CaLCuV TrAP at a reduced affinity in vitro ( <a href="#fig4">Figure 4D–F</a>). These results were consistent with the results of the in vivo Co-IP experiments in which only full-length KYP, but not C-terminal-truncated versions, could recover TrAP ( <a href="#fig3">Figure 3D</a>). Recent structural analysis on KYP revealed that SET and pre-SET domains constitute two modules: one forms a narrow pocket harboring the H3 tail (1–15aa), whereas the other binds the SAM cofactor, together with the post-SET domain ( <a href="#bib14">Du et al., 2014a</a>). Because the post-SET domain does not seem to contribute to the KYP-TrAP interaction, TrAP could potentially occupy the histone-binding cavity ( <a href="#fig4">Figure 4F</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.008"> <span class="label" tagx="label" title="label">Figure 4.</span> </a> </div>   <div class="title" tagx="title" title="title">
TrAP interacts directly with KYP through the SET domain.</div> <p>( <b>A</b>, <b>B</b>) In vitro pull-down assays showed that KYP specifically interacted with TGMV TrAP ( <b>A</b>) and CaLCuV TrAP ( <b>B</b>). Left panel, Coomassie brilliant blue R250 staining of the proteins shows their mobility. Right panel, output of in vitro pull-down assays. All His-MBP-tagged bait proteins and His-GST-tagged prey proteins were purified from <i>Escherichia coli</i> using Ni-NTA columns. In all assays, 2.5 μg of prey proteins were pulled down with the indicated bait proteins (2.5 μg each) using amylose resin. The recovered MBP-tagged bait proteins were monitored by Coomassie brilliant blue R250 staining. All the experiments were done at the same time, and the samples were run in the same gels. The spacers in the images indicate digital rearrangements of the pictures. The output of the GST-tagged prey proteins was analyzed by western blot using a monoclonal anti-GST antibody. ( <b>C</b>) Schematic diagram of full-length and truncated forms of KYP. The numbers on the left refer to the amino acid residues in KYP protein. Locations of SRA, pre-SET, SET, and post-SET domains are shown. ( <b>D</b>) In vitro pull-down assays of truncated KYP proteins and TrAPs. The experiments were done as in ( <b>A</b>, <b>B</b>). The GST negative control was loaded in the lanes marked with (*). ( <b>E</b>) Summary of interaction between the truncated KYP proteins and TrAP encoded by TGMV and CaLCuV. ( <b>F</b>) Model of possible KYP-TrAP interaction based on the experimental results from panels <b>D</b> and <b>E</b>. KYP structure was generated in Chimera from PDB: 4QEN data set ( <a href="#bib15">Du et al., 2014b</a>), domains are color coded and indicated. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.008">http://dx.doi.org/10.7554/eLife.06671.008</a> </p>   </div> <p /> <p>Since the SET domain is well conserved among histone methyltransferases (HMTase) ( <a href="#bib31">Liu et al., 2010</a>), we next investigated whether TrAP interacts with KYP paralogs. In vitro pull-down assays showed that TrAP indeed interacted with numerous tested HMTases (SUVH2, 5, and 6) ( <a href="#fig4s2">Figure 4—figure supplement 2</a>). Notably, loss-of-function mutants of <i>SUVH2</i> display early-flowering phenotype, suggesting that TrAP might target this protein in vivo as well. </p> </div> <div class="trapinhibitsthecatalyticactivityofkypinvitro" title="sec"> <div class="title" tagx="title" title="title">
TrAP inhibits the catalytic activity of KYP in vitro</div> <p>The specific interaction of TrAP with the catalytic SET domain of the SUVHs prompted the question of whether TrAP affects KYP activity. To test this, we set up an in vitro reconstitution of H3K9 methylation using His-GST-KYP purified from <i>Escherichia coli</i> as the enzyme source, recombinant histone 3 as the substrate, and methyl- <sup>14</sup>C-SAM as the methyl donor ( <a href="#fig5">Figure 5A</a>). Under our experimental conditions, 1 µM GST-KYP methylated 3 µM of histone 3 in less than 5 min at 37°C, as detected by saturation of the radioactive signal ( <a href="#fig5">Figure 5B</a>). Excitingly, incubation of His-GST-KYP with His-MBP-TrAP from either TGMV or CaLCuV reduced the initial velocity of KYP transmethylation activity in a dose-dependent fashion, whereas His-MBP alone did not affect KYP catalysis ( <a href="#fig5">Figure 5B</a>). Quantification of signal intensity revealed that the TrAP-KYP molar ratio of 2 was enough to cause approximately 50% inhibition of KYP activity (half maximal inhibitory concentration [IC <sub>50</sub>]) ( <a href="#fig5">Figure 5C</a>). Thus, our results indicated that TrAP potently inhibited the catalytic function of the HMTase. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.011"> <span class="label" tagx="label" title="label">Figure 5.</span> </a> </div>   <div class="title" tagx="title" title="title">
TrAP inhibited HMTase activity of KYP in vitro.</div> <p>( <b>A</b>) Coomassie staining of purified proteins uses for the assays. ( <b>B</b>) In vitro HMTase reconstitution assays with different molar ratio of MBP and MBP-TrAP proteins (0–10×) relative to GST-KYP. The recombinant KYP was incubated without (buffer only) or with the indicated proteins before addition of histone 3 and C <sup>14</sup>-SAM. The reactions were done in a 6-min time course; aliquots were resolved in 18% SDS PAGE and stained with Coomassie blue R250 to show histone 3 input (top panels). The dried gels were auto-radiographed to detect <sup>14</sup>C-methylated histone 3. ( <b>C</b>). Plotting of KYP initial velocity vs TrAP/KYP molar ratio. The initial velocity was calculated from the slope of the linear range in the signal vs time plot for each assay, and then the values were normalized using the non-inhibitor control as a standard of 1 to obtain the relative initial velocity with standard deviation (SD) from at least three biological repeats. The relative initial velocity is plotted as a function of inhibitor:enzyme molar ratio in a logarithmic scale of base 2. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.011">http://dx.doi.org/10.7554/eLife.06671.011</a> </p>   </div> <p /> </div> <div class="trapreducesh3k9me2-repressivemarksinvivo" title="sec"> <div class="title" tagx="title" title="title">
TrAP reduces H3K9me2-repressive marks in vivo</div> <p>Given that in vivo TrAP genetically interferes with the TGS pathway, and in vitro it physically interacts with HMTases (KYP, SUVH2, 5, 6) and inhibits the activity of KYP, we wondered whether TrAP alters KYP function in vivo. To address this question, we conducted chromatin immunoprecipitation (ChIP) analyses of H3K4me3, H3K9me2, and K3K27me3 marks ( <a href="#fig6">Figure 6</a>; <a href="#fig6s1">Figure 6—figure supplement 1</a>; <a href="#fig6s2">Figure 6—figure supplement 2</a>) on numerous KYP-regulated TEs ( <a href="#fig6">Figure 6A</a>) in TrAP overexpression plants as compared to Col-0, <i>ddm1</i>, <i>kyp</i>, and <i>lhp1</i> control plants. Remarkably, all tested loci in <i>TrAP</i> transgenic lines displayed consistent reduction of H3K9me2 and H3K27me3, whereas changes of H3K4me3 were variable. This molecular phenotype mimicked <i>kyp</i> mutant at all tested loci, but not lhp1 mutant. This result indicated inhibitory effect of TrAP on KYP activity on the TEs in vivo. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.012"> <span class="label" tagx="label" title="label">Figure 6.</span> </a> </div>   <div class="title" tagx="title" title="title">
ChIP-qPCR analyses of H3K4me3, H3K9me2, and H3K27me3 in TrAP-regulated loci in vivo.</div> <p>( <b>A</b>) TrAP-activated transposons in heterochromatic regions contained reduced H3K9me2 and H3K27me3 but did not show consistent variation in H3K4me3 marks. ( <b>B</b>) <i>TrAP</i>-upregulated flowering genes showed consistently reduced H3K9me2 and H3K27me3 marks compared to wild-type Col-0. ( <b>C</b>) TrAP-downregulated genes displayed variable changes of H3K9me2 and H3K27me3 marks and no obvious changes of H3K4me3 mark. ( <b>D</b>) Tubulin ( <i>TUB8</i>) was used as internal control for all the ChIP experiments; the percentage enrichment vs input is shown. ChIP assays were conducted on 11-day-old seedlings using antibodies specific for H3K9me2 (Abcam, Cat# ab1220), H3K27me3 (Millipore, Cat# 07-449), and H3K4me3 (Millipore, Cat# 04-745). Enrichment of H3K4me3 and H3K27me3 in each locus is normalized to that of <i>TUB8</i>; H3K9me2 enrichment is plotted as percentage of input. The standard deviation (SD) was calculated from at least three biological repeats. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.012">http://dx.doi.org/10.7554/eLife.06671.012</a> </p>   </div> <p /> <p>Since TrAP is a transcriptional activator protein, we next asked whether the increased transcription of the <i>TrAP</i>-responsive protein-coding genes is associated with changes in the histone methylation status. First, we screened numerous loci that showed transcriptional deregulation in the <i>TrAP</i> overexpression lines ( <a href="#fig2">Figure 2B–D,G,H</a>) for the presence of various histone modifications. We identified a dozen loci in which H3K9me2 marks are easily detected in wild-type plants. We then conducted ChIP-qPCR for these loci in the <i>TrAP</i> transgenic plants. As expected, H3K9me2 and H3K27me3 were reduced in most of the tested loci in <i>ddm1</i>, <i>kyp</i> and <i>lhp1</i> mutants, whereas H3K4me3 was enriched. These results are consistent with the generally antagonistic roles of H3K4me3 and H3K9me2 modifications. Specifically, six out of seven TrAP deregulated loci including the flowering-promoting genes displayed greater than twofold reduction in H3K9me2 and H3K27me3 in the <i>TrAP</i> overexpression plants compared to wild-type Col-0, while changes in H3K4me3 were inconsistent. This scenario was similar to that observed for TEs ( <a href="#fig6">Figure 6B,C</a>). Collectively, the ChIP assays indicated that TrAP interferes with the epigenetic pathways through reducing repressive H3K9me2 marks in vivo. </p> </div> <div class="trapdecreaseschhdnamethylation" title="sec"> <div class="title" tagx="title" title="title">
TrAP decreases CHH DNA methylation</div> <p>KYP, SUVH5, and SUVH6 are required for maintenance of non-CG (CHG and CHH) methylation in <i>Arabidopsis</i> ( <a href="#bib51">Stroud et al., 2013</a>; <a href="#bib50">Stroud et al., 2014</a>). We predicted that inhibition of KYP function by TrAP might indirectly cause reduction in non-CG DNA methylation. To test this, we conducted genome-wide bisulfite sequencing with 11-day-old seedlings of TrAP transgenic plants, <i>kyp</i> mutant, and Col-0. Consistent with previous studies, <i>kyp</i> mutant showed genome wide loss of methylation in CHG (∼42.1%) and CHH (∼21.7%), but not in CG (∼2%) contexts when compared to Col-0. To our surprise, <i>TrAP</i> transgenic plants only exhibited decrease in methylation of CHH (∼11%) but not CHG ( <a href="#fig7">Figure 7A</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.015"> <span class="label" tagx="label" title="label">Figure 7.</span> </a> </div>   <div class="title" tagx="title" title="title">
TrAP reduces CHH DNA methylation in vivo.</div> <p>( <b>A</b>) Genome-wide heatmap of DNA methylation levels in Col-0, <i>kyp</i> mutant, and <i>TrAP</i> transgenic plants (Left). Sequence context of all cytosine, CG, CHG, and CHH methylation was depicted as black, yellow, blue, and red, respectively. The percentage of cytosine methylation is shown for each genotype (Right). ( <b>B</b>) Average distribution of context-specific DNA methylation in Gene- and TE-rich regions in Col-0 (red), <i>kyp</i> mutants (green), and <i>TrAP</i> transgenic plants (blue). ( <b>C</b>). Overlap of CHH DMRs between <i>TrAP</i> transgenic plants and <i>kyp</i> mutant. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.015">http://dx.doi.org/10.7554/eLife.06671.015</a> </p>   </div> <p /> <p>To further analyze the effect of TrAP on DNA methylation, we identified differentially methylated regions (DMRs) by scanning the genomes in 200 bp tiles and comparing the level of methylation among <i>kyp</i> and <i>TrAP</i> overexpression plants with Col-0 (‘Materials and methods’) ( <a href="#SD5-data SD6-data SD7-data">Supplementary files 5, 6, 7</a>). Given that non-CG methylation is highly co-localized and predominant in TE-rich heterochromatic regions ( <a href="#bib51">Stroud et al., 2013</a>, <a href="#bib50">2014</a>; <a href="#bib45">Shen et al., 2014</a>; <a href="#bib17">Dubin et al., 2015</a>; <a href="#bib58">Yang et al., 2015</a>), we separated the DMRs into gene- and TE-rich regions (GRR and TERR respectively); When compared to Col-0, <i>TrAP</i> transgenic plants displayed loss of CHH methylation at GRR but not in TERR, nor in CG or CHG contexts ( <a href="#fig7">Figure 7B</a>). Remarkably, of the 3442 and 1784 GRR hypomethylated DMRs identified in <i>kyp</i> mutant and TrAP transgenic plants, 1642 were shared. To better understand the effect of <i>TrAP</i> expression on CHH DNA methylation, the GRR DRMs were further separated into promoter, terminator, UTR, intronic, and coding regions. We found that <i>TrAP</i> hypomethylated DMRs in coding sequences, UTRs, and introns overlapped almost completely with the <i>kyp</i> mutant (96.7%, 96%, and 95.4% respectively) ( <a href="#fig7">Figure 7C</a>). Notably, gene ontology analysis of the overlapped genes pointed to response genes, specifically in the protein kinase category ( <a href="#fig7s1">Figure 7—figure supplement 1</a>; <a href="#SD7-data">Supplementary file 7</a>). This substantial overlap explains about one-third of the total CHH DMRs in <i>kyp</i> mutant, and one half of <i>kyp</i> DMRs in genic regions ( <a href="#fig7">Figure 7C</a>). Together, these results suggest an inhibitory effect of TrAP in KYP-dependent CHH DNA methylation. </p> </div> <div class="trapinhibitskypactivitytocounterhostdefense" title="sec"> <div class="title" tagx="title" title="title">
TrAP inhibits KYP activity to counter host defense</div> <p>In animals, H3K9 methylation promotes TGS and latency of integrated viruses. In plants, Geminivirus constitutes into a minichromosome that also undergoes epigenetic regulation. The specific TrAP-KYP interaction, inhibition of KYP activity in vitro, and reduction of H3K9me2 and CHH methylation in vivo suggest that KYP is a major factor in combating viruses. Previous studies showed that <i>kyp</i> mutants are hypersusceptible to Geminivirus infection ( <a href="#bib24">Hanley-Bowdoin et al., 2013</a>); these experiments were reproducible in our hands ( <a href="#fig8">Figure 8A–D</a>). Moreover, the hypersusceptibility of <i>kyp</i> mutant to the virus could be rescued by the wild-type <i>KYP</i> transgene under the control of both the native and the constitutive 35S promoters ( <a href="#fig8">Figure 8A–D</a>); further validating a role for KYP in regulation of viral infection. In line with the phenotypic complementation, wild-type <i>KYP</i> almost completely rescued H3K9me2 defects of the TEs in the <i>kyp</i> mutant ( <a href="#fig8">Figure 8E</a>, lanes 1–4). Interestingly, although the accumulation of <i>KYP</i> transcripts was substantially increased when transcribed from the <i>35S</i> promoter when compared to the native promoter, the steady-state protein level was only twofold to threefold higher in the <i>35S-Flag-4Myc-KYP</i> than in the <i>P</i> <sub> <i>KYP</i> </sub>- <i>Flag-4Myc-KYP</i> transgenic plants (data not shown). These observations suggest a possible homoeostatic regulation of this critical TGS component. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.017"> <span class="label" tagx="label" title="label">Figure 8.</span> </a> </div>   <div class="title" tagx="title" title="title">
KYP methylates Geminivirus chromatin as a host defense.</div> <p>( <b>A</b>) Western blot analysis of <i>kyp</i> complementation lines expressing <i>P</i> <sub> <i>KYP</i> </sub>- or <i>35S-Flag-4Myc-KYP</i> using anti-myc antibody. *, a cross-reaction band serves as a loading control. ( <b>B</b>) Representative CaLCuV symptoms with different severities. ( <b>C</b>) Time course of CaLCuV symptom development in <i>kyp</i> mutant and the complementation lines. The mean values were calculated with SD from at least three experiments (&amp;gt;30 plants/line). ( <b>D</b>) CaLCuV symptom severity in Col-0, <i>kyp</i> mutant, and the complementation lines. The mean values were calculated with SD from at least three experiments (&amp;gt;30 plants/line). ( <b>E</b>) ChIP-PCR of H3K9me2 marks in heterochromatic loci in <i>kyp</i> mutant and complementation lines inoculated with mock or CaLCuV. Note: CaLCuV infection largely removed H3K9me2 marks from heterochromatic loci. ( <b>F</b>) Schematic linearized representation of the regions of viral genome A selected for ChIP assays. ( <b>G</b>) ChIP-qPCR assays showed KYP-dependent enrichment of H3K9me2 in several tested loci in the viral chromatin. The relative value of histone methylation in each sample was normalized to that of wild-type control where the signal was arbitrarily assigned a value of 1 with standard deviation (SD) from at least three biological repeats. Note: the region defined by # Primer 7 serves as a negative control. ( <b>H</b>) Western blot analysis to detect Flag-Myc <sub>4</sub>-KYP in the ChIP (IP) samples using anti-myc antibody. ( <b>I</b>) ChIP-PCR assays showed that KYP binds to the viral minichromosome. The ChIP assays were done with a monoclonal anti-Flag antibody. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.017">http://dx.doi.org/10.7554/eLife.06671.017</a> </p>   </div> <p /> <p>Geminivirus minichromosome harbors H3K9me2 ( <a href="#fig8">Figure 8G</a>; <a href="#fig8s1">Figure 8—figure supplement 1</a>) ( <a href="#bib24">Hanley-Bowdoin et al., 2013</a>). Interestingly, H3K9me2 marks were dramatically decreased on several, but not all tested loci in the viral genome in <i>kyp</i> mutant, and the methylation marks were further restored by the <i>KYP</i> transgene, suggesting that viral chromatin H3K9me2 is catalyzed by KYP. Notably, the H3K9me2 amount was substantially enhanced in the locus defined by Primer #9 in the transgenic <i>KYP</i> overexpression plants ( <a href="#fig8">Figure 8G</a>). It is noteworthy that this locus harbors the promoter element for <i>TrAP</i> itself ( <a href="#bib48">Shung and Sunter, 2009</a>). This result suggests that the tight control of <i>TrAP</i> expression likely determines the balance between the host and virus interactions. If so, the result might also explain the relatively milder viral symptoms of <i>35S-Flag-Myc4-KYP</i> lines compared to WT control or <i>P</i> <sub> <i>KYP</i> </sub> <i>-Flag-Myc4-KYP</i> complementation lines ( <a href="#fig8">Figure 8A–D</a>). To further test whether KYP methylated the viral chromatin, we performed ChIP assays using monoclonal anti-Flag antibody to pull down <i>Flag-Myc</i> <sub> <i>4</i> </sub> <i>-KYP</i> bound chromatin ( <a href="#fig8">Figure 8H</a>). Excitingly, KYP was found at all tested loci in the viral chromatin ( <a href="#fig8">Figure 8I</a>). Notably, regions delineated by the primer #7 showed KYP-independent H3K9me2 marks but were still immunoprecipitated in the KYP-chromatin complex. A possible explanation is incomplete chromatin shearing of the small viral mini-chromosome under the conditions used in this experiment, which were standardized for host chromatin ChIP. Alternatively, this might suggest the presence of additional epigenetic regulation that masks KYP activity on this locus. This notwithstanding, our results indicate that KYP directly deposits the H3K9me2 mark on the Geminivirus minichromosome to reinforce the silent status of the virus. </p> <p>Since constitutive expression of <i>TrAP</i> reduced H3K9me2 in vivo, we wondered if Geminivirus infection could also decrease the repressive marks in the host. To this end, we tested the KYP-controlled endogenous transposons for H3K9me2 accumulation. CaLCuV-infected plants showed 60–100% H3K9me2 loss in the tested loci compared to the amount in the mock-inoculated plants ( <a href="#fig8">Figure 8E</a>), this is reminiscent of the molecular phenotype of <i>TrAP</i> transgenic plants, which showed lower enrichment of H3K9me2 mark at the studied loci ( <a href="#fig6">Figure 6C</a>). These results indicated that Geminivirus infection largely removed the repressive H3K9me2 marks in the transposons, and that the removal resulted at least in part from TrAP function. Given that TrAP is known to activate the expression of viral genes in the minichromosome and that endogenous transposons can serve as a proxy for viral genomes, it is conceivable that virus-encoded TrAP acts to suppress KYP in order to prevent the deposition of H3K9me2-repressive marks in the epigenome, to activate the expression of viral genes. </p> </div> <div class="kypmutantssustainsystemicinfectionofcalcuvlackingafunctionaltrapgene" title="sec"> <div class="title" tagx="title" title="title"> <i>kyp</i> mutants sustain systemic infection of CaLCuV lacking a functional <i>TrAP</i> gene </div> <p>Previous studies show that TrAP is required for the accumulation of the virus infective form, single-stranded (ss) DNA ( <a href="#bib25">Hayes and Buck, 1989</a>). TrAP is indispensable for systemic infection of Begomoviruses because it activates the expression of the viral ssDNA binding proteins (nuclear shuttle protein and coat protein), which are essential for releasing the virus from the nucleus and for cell-to-cell spreading ( <a href="#bib49">Sunter and Bisaro, 1992</a>). If TrAP-mediated transcriptional activation and accumulation of ssDNA result from inhibition of KYP and correspondingly heterochromatin formation, then infectivity of CaLCuV lacking TrAP should be impaired in wild-type but recovered in <i>kyp</i> plants. To test this hypothesis, we engineered a CaLCuV variant without functional TrAP protein (CaLCuV Δ <i>trap</i>) by changing a single nucleotide (T to A) that produces an amber mutation and introduces an XbaI restriction site after the sixth codon in the <i>TrAP</i> gene ( <a href="#fig9">Figure 9A</a>). Then, we assessed systemic infection of CaLCuV and CaLCuV Δ <i>trap</i> on wild-type and <i>kyp</i> plants ( <a href="#fig9">Figure 9B</a>). Consistent with the hypersusceptible phenotype of <i>kyp</i> mutants to CaLCuV ( <a href="#fig8">Figure 8C,D</a>), these plants accumulated significantly higher titers of CaLCuV relative to wild-type control as both the replicative intermediate, open circle (OC), and the infective particle, SS DNA ( <a href="#fig9">Figure 9C</a>). Consistent with previous reports, the wild-type plants did not show any symptoms of infection ( <a href="#fig9">Figure 9E</a>) or systemic accumulation of CaLCuV Δ <i>trap</i>, as evidenced by Southern blot analyses and PCR ( <a href="#fig9">Figure 9C</a>). Particularly, PCR amplification followed by XbaI digestion of the viral <i>TrAP</i> region in wild-type plants infected with CaLCuV Δ <i>trap</i> showed loss of the XbaI restriction site ( <a href="#fig9s1">Figure 9—figure supplement 1</a>). Excitingly, the viral DNA was detected, in low amount but reproducibly, in <i>kyp</i> mutants. Further PCR amplification of the <i>TrAP</i> gene followed by XbaI digestion, confirmed that the detected viral DNAs in <i>kyp</i> mutants indeed came from CaLCuV Δ <i>trap</i> and not from a reversion of the mutation in the virus genome. These results show that <i>kyp</i> plants sustained systemic infection of CaLCuV Δ <i>trap</i>. Noticeably, the titers of accumulated CaLCuV Δ <i>trap</i> were much lower than the ones of CaLCuV, suggesting either redundant activity of <i>KYP</i> paralogs in host defense or necessity of additional functions of TrAP besides inhibition of KYP activity for the virus to achieve efficient infection. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.7554/eLife.06671.019"> <span class="label" tagx="label" title="label">Figure 9.</span> </a> </div>   <div class="title" tagx="title" title="title">
Infectivity of CaLCuV lacking functional TrAP protein.</div> <p>( <b>A</b>) Sequence alignment of CaLCuV Δ <i>trap</i> and CaLCuV sequences. The translated amino acids are shown for each sequence, and the XbaI restriction site resulting from the T to A point mutation is highlighted. ( <b>B</b>) Schematics of the systemic infection experiment. Plants with eight true leaves (depicted in gray) were inoculated with the begomovirus, and 18 days post inoculation nine to 11 newly emerged, not inoculated, rosetta leaves (depicted in bright green) were collected to test for virus systemic infection. ( <b>C</b>) Southern blot analysis of viruses in non-inoculated leaves of infected plants. Ethidium bromide staining of total genomic DNA serves as a loading control (top panels). Southern blots were probed against CaLCuV DNA A common region (CR) (bottom panels); the viral populations are indicated as the replicative intermediate open circle (OC), linear (L), super coiled (SC) and the infective particle ss DNA. ( <b>D</b>) Genotypic confirmation of the systemically amplified viruses. Top panels show PCR amplification of a <i>TrAP</i>-containing region; bottom panels show EcoRI and XbaI digestions of PCR products to examine the presence of the amber mutation in the gene. ( <b>E</b>) Exemplary phenotypes of wild-type and <i>kyp</i> mutant plants inoculated with mock, CaLCuV and CaLCuV Δ <i>trap</i>. ( <b>F</b>) Model of TrAP suppression of KYP activity to prevent epigenetic silencing of the viral chromatin. Geminivirus genome is packed on histone octamers to form a minichromosome. The minichromosome undergoes extensive H3K9me2 modification deposited by host-encoded KYP, and this modification could be further reinforced by DNA methylation, leading to formation of viral heterochromatin. As a counter-defense strategy, Geminivirus-encoded TrAP protein inhibits KYP activity to maintain the euchromatic status of the minichromosome to allow active replication and transcription of viral genes and correspondingly to escape host surveillance. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.019">http://dx.doi.org/10.7554/eLife.06671.019</a> </p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>TrAP was among the first viral suppressors identified to interfere with the TGS pathway. The prevailing model is that TrAP lowers the reservoir of transferable methyl groups by targeting ADK, a key enzyme in the SAM pathway. Here, we propose a novel model in which TrAP regulates TGS by directly targeting KYP ( <a href="#fig9">Figure 9F</a>). Several pieces of evidence supported our notion: (1) TrAP genetically interfered with the TGS pathway ( <a href="#fig2">Figure 2</a>); (2) TrAP directly interacted with KYP in vivo and also with other HMTase <i>s</i> in vitro ( <a href="#fig3">Figure 3</a>; <a href="#fig4">Figure 4</a>); (3) TrAP inhibited the catalytic activity of KYP in vitro ( <a href="#fig5">Figure 5</a>); (4) TrAP reduced the repressive H3K9me2 marks in vivo ( <a href="#fig6">Figure 6</a>), and correspondingly, reactivated numerous loci that are otherwise repressed by KYP ( <a href="#fig2">Figure 2</a>); (5) TrAP decreased CHH methylation in gene-rich regions that are also regulated by KYP ( <a href="#fig7">Figure 7</a>); (6) methylation of viral chromatin entailed KYP ( <a href="#fig8">Figure 8</a>); (7) KYP bound the viral chromatin ( <a href="#fig8">Figure 8</a>); and (8) <i>kyp</i> mutants but not wild-type plants sustain low systemic infection of CaLCuV lacking TrAP protein ( <a href="#fig9">Figure 9</a>). To our best understanding, this is the first evidence that a viral protein directly suppresses HMTases in the host TGS machinery. </p> <p>Given that TrAP protein could concurrently limit the upstream supply of the methyl groups and directly inhibit downstream enzymatic activity of KYP, Geminiviruses appear to have evolved sophisticated strategies to cripple the host TGS pathway. What would be the biological advantages of blocking the TGS pathways? In eukaryotes, chromatin appears to be a critical battleground for virus–host interaction. Animals use histone modifications to reinforce the latency of integrated viruses ( <a href="#bib13">du Chene et al., 2007</a>; <a href="#bib38">Narasipura et al., 2014</a>; <a href="#bib39">Park et al., 2014</a>). Plant DNA viruses including Geminiviruses and pararetroviruses replicate as nuclear minichromosomes or episomes. Clearly, TGS functions as an immune system to control virus replication and the expression of viral genes, in a similar fashion as the repression of endogenous TEs and transposon remnants. In previous studies and here in our experiments, <i>kyp</i> mutants displayed hypersusceptibility to Geminivirus infection, which could be rescued by exogenous wild-type <i>KYP</i> gene. KYP directly acts on viral chromatin and deposits the repressive H3K9me2 mark on the viral chromatin. Moreover, viruses lacking the essential TrAP protein can, although inefficiently, cause systemic infection in <i>KYP</i>-deficient hosts. All these facts point out the unambiguous role of KYP-mediated TGS in defense against viral infection. On the other hand, TrAP functions to inhibit KYP catalytic activity, reducing the repressive H3K9me2 mark, to activate transcription of viral genes. Our KYP reconstitution assays show that TrAP:KYP molar ratio of around 2 is enough to cause ∼50% inhibition of KYP activity, indicating that TrAP is a potent inhibitor of HMTase activity. Thus, direct inhibition of KYP represents a novel counter-defense mechanism for virus survival in the hosts. This mechanism could account for the long-documented essential role of TrAP in expression of viral genes including the coat protein and the nuclear shuttle protein ( <a href="#bib49">Sunter and Bisaro, 1992</a>; <a href="#bib59">Yang et al., 2007</a>; <a href="#bib46">Shen et al., 2009</a>; <a href="#bib24">Hanley-Bowdoin et al., 2013</a>). </p> <p>In plants, H3K9me2 marks often correlate with non-CG DNA methylation, and in particular with CHG modification. The tight coordination results from a self-enforcing loop consisting of KYP and CMT3 ( <a href="#bib14">Du et al., 2014a</a>). Briefly, KYP methylates H3K9 to generate the binding sites for CMT3, which further methylates CHG DNA to create more binding sites for KYP. Consequently, the crosstalk between DNA and histone methylation ascertains the silent status of heterochromatin. We hypothesized that TrAP targets KYP to reduce H3K9me2 marks, and this inhibition would further decrease DNA methylation both on the host and on the viral genome. Here, we found that ectopic expression of TrAP had no effect on CHG but only on CHH methylation, at sites that are up to 96.7% overlapped with the hypo-methylated regions in <i>kyp</i> mutant ( <a href="#fig7">Figure 7C</a>). In plants, sequence contexts of CHG and CHH methylation are largely overlapped throughout the genome and are maintained to a limited extent by all non-CG methyltransferases ( <a href="#bib50">Stroud et al., 2014</a>). One outstanding question would be how could TrAP differentially alter methylation of CHH rather than CHG, given that TrAP targets KYP? Methylation in the CHH context is catalyzed by CMT2 and DRM1/2 in <i>Arabidopsis</i>, and the sites regulated through the two sets of enzymes are mostly non-overlapping. Thus, the functions of CMT2 and DRM1/2 are mostly non-redundant at CHH sites ( <a href="#bib51">Stroud et al., 2013</a>, <a href="#bib50">2014</a>). DRM1/2 largely catalyze CHH methylation at the TE-rich regions ( <a href="#bib50">Stroud et al., 2014</a>). In our study, we did not observe any effect of TrAP on CHH methylation in the TEs, suggesting that TrAP might not (or not sufficiently) interrupt the RdDM pathway that entails DRM1/2 and 24-nt siRNAs ( <a href="#bib50">Stroud et al., 2014</a>). Importantly, it has been recently discovered that bulk CHH methylation is maintained by CMT2 ( <a href="#bib45">Shen et al., 2014</a>; <a href="#bib50">Stroud et al., 2014</a>; <a href="#bib17">Dubin et al., 2015</a>) and that its activity is largely dependent on H3K9me2. In this scenario, CMT2 recognizes methylated H3K9 but preferentially binds to di-methylated over mono-methylated histone tails. This preference towards H3K9me2 is not observed in the CHG methyltransferase CMT3, which can equally bind to all forms of H3K9 methylation ( <a href="#bib50">Stroud et al., 2014</a>). We envision that methylation at CHH sites could be more sensitive to changes in H3K9me2 than CHG, since CMT3 could still maintain CHG DNA methylation in the presence of H3K9me1 resulting from residual activity of HMTase, as would be the case when KYP is inhibited by TrAP. Alternatively, CHH methylation might play an important but yet unappreciated regulatory role in host defense genes ( <a href="#fig7s1">Figure 7—figure supplement 1</a>). If so, preferentially targeting these loci by TrAP protein might represent a new counter-defense mechanism. </p> <p>In the host, Geminivirus DNAs also undergo extensive methylation modification ( <a href="#bib42">Raja et al., 2008</a>). In our study, TrAP reduces CHH methylation in the host and possibly in the viral genome, too. Remarkably, the Geminivirus replicase, Rep/AC1/C1 and the embedded protein AC4/C4, downregulate the expression of host DNA methyltransferases <i>MET1</i> and <i>CMT3</i> ( <a href="#bib40">Pumplin and Voinnet, 2013</a>; <a href="#bib44">Rodriguez-Negrete et al., 2013</a>); hence, it interrupts the reinforcing loop of histone and DNA methylation. Consistent to this study, <i>cmt3</i> mutants exhibit hypersusceptibility to viral infection ( <a href="#bib42">Raja et al., 2008</a>). The fact that Rep/C4 repress <i>CMT3</i> expression in the host is in perfect alignment with our ChIP assay results on CaLCuV-infected plants, where the loss of H3K9me2 is even more severe than the observed in TrAP transgenic plants. Thus, interruption of the compelling feedback loop of histone and DNA methylation represents an important strategy to sustain transcription of viral chromatin. Together, it seems that the synergistic inhibition of histone and DNA methyltransferases by Geminivirus proteins evolves as a powerful tactic to win the arms race between host and pathogen. </p> <p>Our in vitro assays clearly demonstrated that TrAP predominantly binds to the catalytic domain of KYP and inhibited its enzymatic activity. Whether TrAP might alter KYP conformation or block the accessibility of substrates to the active sites upon interaction awaits future structural analysis. We note, however, that <i>Arabidopsis</i> has 49 SET domain containing proteins of which 31 are considered to have HMTase activity ( <a href="#bib31">Liu et al., 2010</a>). Arising from this fact is whether TrAP specifically targets KYP or promiscuously acts on additional HMTases. Although TrAP overexpression plants display molecular features in common with <i>kyp</i> mutant (i.e., reduced H3K9me2 levels), and an early flowering phenotype similar to loss-of-function mutant of the <i>kyp</i> paralog, <i>SUVH2</i>, the transgenic plants are phenotypically different from <i>kyp</i> single mutant. This could be due to functional redundancy between <i>KYP</i> and its paralogs in vivo; alas, the morphological phenotype of the higher-order mutants has not been fully documented. </p> <p>We also noticed that <i>TrAP</i>-overexpression plants are morphologically similar to <i>lhp1,</i> which is characteristic of the H3K27me3 pathway. Moreover, the substantial overlapping of <i>TrAP</i>-responsive genes with <i>lhp1</i>-regulated genes strongly suggested that TrAP might target the <i>LHP1</i>-related H3K27me3 pathway ( <a href="#bib66">Zheng and Chen, 2011</a>). Indeed, most of our tested loci in the host genome exhibited decreased H3K27me3 levels, consistent with the fact that ∼50% genes were co-regulated by TrAP and <i>LHP1</i> in a genome-wide scale. Interestingly, recent ChIP–chip studies revealed that H3K27me3 and <i>LHP1</i>-bound sites are predominantly distributed in the euchromatic regions ( <a href="#bib53">Turck et al., 2007</a>; <a href="#bib60">Zhang et al., 2007a</a>, <a href="#bib62">2007b</a>). This distribution is not correlated with KYP-dependent H3K9me2 marks that are highly enriched at pericentromeric regions as large and uninterrupted heterochromatic blocks ( <a href="#bib31">Liu et al., 2010</a>; <a href="#bib5">Black et al., 2012</a>; <a href="#bib14">Du et al., 2014a</a>; <a href="#bib32">Liu et al., 2014</a>). H3K9me2 can also occur in euchromatic regions but rather exist as small heterochromatin patches ( <a href="#bib66">Zheng and Chen, 2011</a>). In our current study, we did not further examine whether TrAP physically targets H3K27me3 HMTases in the <i>LHP1</i> pathway. But it is plausible that TrAP inhibits both KYP and <i>LHP1</i> pathways. Notably, we did not observe the repression of transcriptionally active histone methylation marks, such as H3K4me3 and H3K36me3. This observation suggests that TrAP does not target the HMTases that are writers for the active marks. How can TrAP distinguish KYP from those active writers remains unclear, but we propose that structural components other than the SET domain might contribute to the recognition and affinity of TrAP binding to HMTases ( <a href="#fig4">Figure 4</a>). Alternatively, additional cellular factors might also contribute to the specificity. </p> <p>The direct consequence of TrAP-dependent inhibition of KYP activity is to activate viral transcription and replication. Because KYP is a key effector of TGS in the host and regulates a broad array of endogenous genes, interference with this core component would reprogram the expression profile of the host genome and thus trigger a series of downstream cascade signaling events that impact the balance of host/virus interaction. As an example, TrAP suppresses auxin and cell growth, whether this change might constitute a defense mechanism for the host benefit or create a favorable cellular niche for virus propagation remains for further investigation. </p> <p>In conclusion, our results support the notion that Geminivirus-encoded TrAP protein interferes with the TGS pathway and abrogates epigenetic silencing by direct interaction with KYP and inhibition of its transmethylation activity. Thus, Geminvirus TrAP functions clearly different from most of previously characterized viral suppressors, which target various steps of the PTGS pathway ( <a href="#bib12">Ding and Voinnet, 2007</a>). Together with previous studies ( <a href="#bib42">Raja et al., 2008</a>), we provide evidence that KYP evolves as a critical immune system to control invading nucleic acids in plants; this is reminiscent of the roles of human SUV39H1 in maintaining the latency of HIV ( <a href="#bib13">du Chene et al., 2007</a>), Epsteinn-Barr virus (Imai et al., 2014), and some other mammalian viruses. Thus, sequestering or interfering with this core component is an effective strategy for Geminivirus to block TGS and to subvert host defense; we expect this strategy to be used by other suppressors in plants and mammals. Given that many viral suppressors interrupt different steps in the PTGS pathways to efficiently combat host surveillance, it would not be surprising that additional key cellular factors in the TGS pathway, besides HMTase, were readily targeted by invading viruses in eukaryotes. Importantly, it is believed that compounds or drugs that alter chromatin methylation might ultimately be the most effective means of combating disease. Our discovery that TrAP inhibits a histone-modifying enzyme also offers a new natural strategy to develop epigenetic-targeted drugs to cure human diseases that arise from epigenetic dysfunction ( <a href="#bib25a">Højfeldt et al., 2013</a>) or to engineer new biotechnological products to improve agricultural productivity. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="dnaconstructions" title="sec"> <div class="title" tagx="title" title="title">
DNA constructions</div> <p>All the plant constructs were made using the Gateway system (Invitrogen, Carlsbad, CA, United States) ( <a href="#bib61">Zhang et al., 2005</a>). The destination vectors (containing the destination cassette–DC-) pHyg-DC-CFP, pBA-DC-CFP, pBA-DC-3HA, pBA-DC, pBA-Flag-4Myc-DC, and pER10-YFP-DC ( <a href="#bib64">Zhang et al., 2006b</a>) were used for transient expression in <i>N. benthamiana</i> and for stable <i>Arabidopsis thaliana</i> transformation. The vector pER10 for betaestradiol-inducible expression under the XVE promoter ( <a href="#bib68">Zuo et al., 2000</a>) was modified to obtain the destination vector pER10cLUC-3HA-DC. For this, the pER10 vector was linearized with XhoI, filled in with Klenow fragment; and further digested with PacI. In parallel, the cLuc-3HA-DC insert was obtained by linearizing the pCambia1300-cLuc-3HA-DC ( <a href="#bib65">Zhang et al., 2011</a>) plasmid treated with SacI/Klenow treatment and further digested with PacI. The vector and insert were ligated and transformed into <i>E. coli</i> DB3.1 and finally confirmed by sequencing. This vector was used together with pCambia Myc-DC-nLUC ( <a href="#bib65">Zhang et al., 2011</a>) for transient expression in <i>N. benthamiana</i>. </p> <p>The cDNA or DNA fragments were cloned into pENTR/D vectors, confirmed by sequencing, and then transferred to the appropriate destination vectors by recombination using the LR Clonase (Invitrogen). The primers for the cloning are listed in the <a href="#SD8-data">Supplementary file 8</a>. To drive KYP expression from its native promoter, we amplified a 2.7-Kb genomic region immediately upstream of the <i>KYP</i> start codon with the primers P <sub>KYP</sub> EcoRV for and P <sub>KYP</sub> BamHI rev. The binary vector pBA002a Flag-4Myc-KYP was obtained by the Gateway system using LR clonase. The resultant plasmid and PCR product harboring <i>KYP</i> promoter were digested with EcoRV/BamHI and ligated with T4 DNA ligase. The plasmid was confirmed by sequencing using the primer P <sub>KYP</sub> seq For. </p> <p>For expression of recombinant proteins, the cDNA or DNA fragments were cloned into pMCSG9 or pMCSG10 vectors to produce His-MBP- or His-GST-tagged proteins respectively, by ligation independent cloning ( <a href="#bib18">Eschenfeldt et al., 2009</a>) using primers that include 18 nt identical to the ends of the linearized pMCSG vector ( <a href="#SD8-data">Supplementary file 8</a>). The pMCSG plasmids were linearized with the blunt end restriction enzyme SspI, and the sticky ends were generated by T4 DNA Polymerase (NEB, Ipswich, MA, United States) supplemented with 2.5 mM dGTP. In parallel, the complementary sticky ends in the PCR products were generated supplementing the T4 DNA Polymerase (NEB) with 2.5 mM dCTP. The mixture of treated plasmid:PCR (3:4, respectively) was incubated on ice for 30 min to allow annealing of the complementary free strands and transformed in <i>E. coli</i> DH5α. The plasmids were further confirmed by sequencing and transformed into <i>E. coli</i> BL21 Rossetta DE3 for expression. </p> <p>To engineer the CaLCuV Δ <i>trap</i> DNA A infective clone the pNSB1090 plasmid was subjected to site directed mutagenesis by amplification with the primers CaLCuV_AL2_null_XbaI_for and CaLCuV_AL2_null_XbaI_rev ( <a href="#SD8-data">Supplementary file 8</a>), digested overnight with DpnI, cleaned with the QIAquick PCR Purification Kit (Qiagen, Germantown, MD, United States), and amplified in <i>E. coli</i> DH5α. The plasmid was confirmed by sequencing and transformed in <i>Agrobacterium tumefasciens</i> ABI for the virus infection assays. </p> </div> <div class="transgenicplants" title="sec"> <div class="title" tagx="title" title="title">
Transgenic plants</div> <p> <i>A. thaliana</i> (Col-0) plants were transformed with binary vectors by the floral-dip method ( <a href="#bib9">Clough and Bent, 1998</a>; <a href="#bib63">Zhang et al., 2006a</a>). The transgenic seeds were selected on standard MS medium ( <a href="#bib36">Murashige and Skoog, 1962</a>) containing the appropriate selective agents: 10 mg/l glufosinate ammonium (Sigma-Aldrich, St. Louis, MO, United States) or 25 mg/l hygromycin (Sigma), together with 100 mg/l carbenicillin (Sigma). <i>kyp</i> mutant (SALK_130630C) was obtained from the Arabidopsis Biological Resource Center (ABRC) and confirmed by genotyping and qRT-PCR. </p> </div> <div class="expressionandpurificationofrecombinantproteins" title="sec"> <div class="title" tagx="title" title="title">
Expression and purification of recombinant proteins</div> <p>For in vitro pull-down or HMTase reconstitution assays, the recombinant proteins were purified following either one or two-step affinity purification procedure. His-MBP-tagged proteins were first purified through Immobilized Metal Affinity Chromatography (IMAC) using the Ni-NTA resin (Qiagen), followed by amylose resin (NEB) according to manufacturers' protocols. His-GST-tagged proteins were initially purified through sepharose glutathione column (GE Healthcare, United Kingdom) followed by IMAC. Specifically for HMTase assays, His-MBP-CaLCuV_TrAP, His-MBP-TGMV_TrAP, and His-MBP were prepared in lysis buffer (50 mM Tris-HCl pH 9, 300 mM NaCl, 10 mM 2-mercaptoethanol, 2 mM PMSF), incubated with the Ni-NTA resin at 4°C for 1 hr, eluted with 300 mM imidazole and immediately incubated with the amylose resin (NEB) at 4°C for 1 hr. The proteins were eluted with 10 mM maltose and the elute was further separated by size exclusion chromatography (SEC) in column buffer (20 mM Tris-HCl pH 9, 100 mM NaCl); the fractions containing the target protein were pulled together, concentrated to 100 µM, aliquoted and stored at −80°C until usage. His-GST-KYP was prepared in PBS buffer (140 mM NaCl, 2.7 mM KCl, 10 mM Na <sub>2</sub>HPO <sub>4</sub>, 1.8 mM KH <sub>2</sub>HPO <sub>4</sub>, pH 7.3, 10 mM 2-mercaptoethanol) incubated with the sepharose glutathione 1 hr at 4°C, eluted with elution buffer (50 mM Tris-HCl pH 9, 200 mM NaCl, 10 mM reduced glutathione, 10 mM 2-mercaptoethanol). The elute was further purified through the Ni-NTA column and finally through SEC in column buffer (20 mM Tris-HCl pH 9, 100 mM NaCl); the fractions containing the target protein were pulled together, concentrated to 25 μM and aliquoted for usage. </p> <p>For in vitro pull down assay, both the prey (His-GST-CaLCuV-TrAP, His-GST-TGMV-TrAP, and His-GST) and the bait (His-MBP-Bait) proteins were purified by IMAC using the lysis buffer (50 mM Tris-HCl pH 8, 300 mM NaCl, 20 mM imidazole, 2 mM PMSF), incubated with the Ni-NTA resin at 4°C for 1 hr, eluted with 300 mM imidazole and immediately dialyzed in storage buffer (20 mM Tris-HCl pH 8, 150 mM NaCl, 2 mM 2-mercaptoethanol, 50% Glycerol) at 4°C overnight. </p> </div> <div class="invitropulldownandco-ipassays" title="sec"> <div class="title" tagx="title" title="title">
In vitro pull down and Co-IP assays</div> <p>In vitro pull-down assays and in vivo Co-IP were done as described ( <a href="#bib61">Zhang et al., 2005</a>). Briefly, 2.5 μg of 6His-GST-tagged prey proteins were pre-absorbed to 50 µl of the amylose resin (NEB) for 1 hr at 4°C in 1 ml of binding buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 0.2% glycerol, 0.6% Triton X-100, 0.5 mM 2-mercaptoethanol, 2 mM PMSF). The proteins were recovered by ultracentrifugation at 12,000× <i>g</i> for 2 min, transferred to a second tube containing 2.5 µg of the MBP-tagged bait protein, and incubated at room temperature for 2 hr. The protein complexes were harvested by adding 50 µl amylose resin beads, followed by 2 hr incubation at room temperature, and cleaned with six vigorous washes with buffer. The pulled-down proteins were resolved by SDS-PAGE and the preys were detected by western blot using anti-GST antibody. </p> <p>For Co-IP experiments, <i>N. benthamiana</i> leaves were collected 2 days after agroinfiltration, ground in liquid nitrogen and stored at −80°C until use. For the assay, total proteins were extracted from 0.4 g of ground powder in 1.2 ml (3 vol) of IP buffer (40 mM Tris-HCl pH 7.5, 300 mM NaCl, 5 mM MgCl <sub>2</sub>, 2 mM EDTA, 4 mM DTT, 0.5% Triton X-100, 1 mM PMSF, 5% glycerol, 1 pellet/25 ml Complete EDTA-free protease inhibitor [Roche, Indianapolis, IN, United States]); then, the soluble proteins were cleared twice by ultracentrifugation at 20,000 × rcf for 15 min at 4°C. The protein complexes were immunoprecipitated with 15 µl Anti-c-Myc-agarose affinity gel (Sigma–Aldrich #A7470) at 4°C for 2 hr, the unspecific-bound proteins were removed by four consecutive washes with the IP buffer with 10 min incubation each at 4°C. The protein complexes were eluted in 200 μl of elution buffer (5 mM EDTA, 200 mM NH <sub>4</sub>OH) for 20 min. The supernatant was collected, frozen in liquid nitrogen and dried using the Savant SpeedVac concentrator; finally, the sample was solubilized in 50 μl of 2× SDS-loading buffer for western blot analyses. </p> </div> <div class="two-stepimmunoprecipitationformassspectrometryanalysis" title="sec"> <div class="title" tagx="title" title="title">
Two-step immunoprecipitation for mass spectrometry analysis</div> <p>9-day-old wild-type control and <i>Arabidopsis</i> transgenic plants expressing <i>XVE-Flag-Myc4-TrAP</i> were induced for 16 hr with 25 µM ß-estradiol in liquid MS media, ground in liquid nitrogen and stored at −80°C until use. For the assay, total proteins were extracted from 10 g of ground powder in 40 ml (4 vol) of IP buffer (20 mM Tris-HCl pH 7.5, 150 mM NaCl, 4 mM MgCl <sub>2</sub>, 50 μM ZnCl <sub>2</sub>, 0.1% Triton X-100, 1 mM PMSF, 1% glycerol, 1 pellet/17 ml Complete EDTA-free protease inhibitor [Roche], 15 μM MG132); then, the soluble proteins were cleared twice by ultracentrifugation at 20,000 × rcf for 15 min at 4°C. The protein complexes were first immunoprecipitated using 500 μl of Anti-FLAG M2 magnetic beads (Sigma-Aldrich, Cat# M8823) and incubated in slow rotation for 2 hr at 4°C, the nonspecific-bound proteins were removed by three consecutive washes with 15 ml of IP buffer for 10 min incubation each at 4°C. The protein complexes were then eluted by competition with 100 mg/ml FLAG peptide and subsequently immunoprecipitated with 100 µl Anti-c-Myc-agarose affinity gel (Sigma–Aldrich #A7470) at 4°C for 1.5 hr, the nonspecific-bound proteins were removed by five consecutive washes with the IP buffer with 5 min incubation each at 4°C. The protein complexes were eluted in 200 μl of elution buffer (5 mM EDTA, 200 mM NH <sub>4</sub>OH) for 20 min. The supernatant was collected, frozen in liquid nitrogen and dried using the Savant SpeedVac concentrator; finally, the sample was solubilized in 30 μl of 2× SDS-loading buffer and run in 10% SDS-PAGE. The samples were run to one-third of the gel, stained with Coomassie blue and collected by excising the whole lane for mass spectrometry analysis in the Taplin Mass Spectrometry Facility at Harvard Medical School. </p> </div> <div class="southernblotanalyses" title="sec"> <div class="title" tagx="title" title="title">
Southern blot analyses</div> <p>The plant material was lysed in CTAB buffer (100 mM Tris HCl pH 8.0, 20 mM EDTA pH 8.0, 1.4 M NaCl, 2% CTAB, 2% ß-mercaptoethanol); then total DNA was extracted with phenol:chlorophorm:isoamyl alcohol (25:24:1) and precipitated with 2-propanol. The DNA was treated with RNase A and further purified with phenol:chlorophorm:isoamyl alcohol (25:24:1) and precipitated with ethanol, then dissolved in ultrapure water. The specified amount of DNA was separated by electrophoresis in 0.8% agarose, transferred overnight by capillarity to a Hybond-N membrane (GE Healthcare), and probed with <sup>32</sup>P-labeled probe targeting the CR region of CaLCuV DNA A. The probe was obtained by PCR using the primers CR_F and CR_R ( <a href="#SD8-data">Supplementary file 8</a>) and labeled using the Rediprime II DNA Labeling System (GE Healthcare) following the manufacturer's instructions. </p> </div> <div class="rnablotanalyses" title="sec"> <div class="title" tagx="title" title="title">
RNA blot analyses</div> <p>Total RNA was extracted using Trizol reagent from either adult rosette leaves or 2-week-old seedlings of independent transgenic lines, the RNA blots were then performed as previously described ( <a href="#bib64">Zhang et al., 2006b</a>). </p> </div> <div class="immunoprecipitationofflag-ago1-associatedsmallrnas" title="sec"> <div class="title" tagx="title" title="title">
Immunoprecipitation of Flag-AGO1-associated small RNAs</div> <p>Immunoprecipitation of Flag-AGO1-associated small RNAs were performed as described ( <a href="#bib64">Zhang et al., 2006b</a>). RNA was recovered with Trizol reagent from the immunoprecipitates, separated in 8 M urea, 15% polyacrylamide gels and subjected to RNA blot analysis of low-molecular-weight RNAs. </p> </div> <div class="luciferasecomplementationimaging(lci)assay" title="sec"> <div class="title" tagx="title" title="title">
Luciferase complementation imaging (LCI) assay</div> <p>The LCI was performed on 4-week-old <i>N. benthamiana</i> leaves infiltrated with various combinations of <i>A. tumefaciens</i> GV3101 harboring pCambia Myc-TrAP-nLUC or pCambia Myc-nLUC and <i>A. tumefaciens</i> ABI carrying pER10cLUC-3HA or pER10cLUC-3HA-candidate proteins. The agrobacteria containing the pER10 plasmids were incubated with 25 µM beta-estradiol for 3 hr prior infiltration, and all the cultures were adjusted to OD <sub>600</sub> = 0.8. The transfected leaves were assayed 2 days after agroinfiltration by adding the substrate (10 mM luciferin). The sprayed leaves were incubated in total darkness for 5 min and photographed using an electronmultiplying charge-coupled device (EMCCD) camera, Cascade II 512, from Photomerics (Roper Scientific, Trenton, NJ, United States). The images were processed with WinView32 Ver 2.5.19.7 (Roper Scientific). </p> </div> <div class="confocalmicroscopyandfretassays" title="sec"> <div class="title" tagx="title" title="title">
Confocal microscopy and FRET assays</div> <p>Leaves of 4-week-old tobacco plants ( <i>N. benthamiana</i>) were agroinfiltrated with syringe without needle as previously described ( <a href="#bib61">Zhang et al., 2005</a>) with <i>A. tumefasciens</i> ABI carrying pBA-TrAP-CFP and pER10-YFP-Test protein. The agrobacteria containing the pER10 plasmids were incubated with 25 µM beta-estradiol for 3 hr prior infiltration, and all the cultures were adjusted to OD <sub>600</sub> = 0.8. The plants were maintained for 2 days at 24°C (16 hr light/8 hr dark). The co-localization was evaluated using a Nikon inverted microscope Eclipse T <i>i</i>-E (Nikon, Japan). CFP signal was measured by excitation with Shutter 10-3 filter 3 (CFPHQ [Ex]), and emission was detected at 485 nm; YFP used Shutter 10-3 filter 4 (YFPHQ [Ex]) and emission was detected at 540 nm. The images were processed using NIS-Elements-AR 4.30.01 (Nikon) and Adobe Photoshop software. </p> <p>FRET-AB experiments were performed on <i>N. benthamiana</i> epidermal cells of 4-week-old leaves agroinfiltrated with a 1:1 mixture of pBA-TrAP-CFP and pER10-YFP-KYP to a final OD <sub>600</sub> = 0.8. YFP and CFP signals were captured with a Zeiss LSM 710 confocal Microscope (Zeiss, Germany). FRET was determined by the acceptor photobleaching method ( <a href="#bib26">Kenworthy, 2001</a>; <a href="#bib10">Daelemans et al., 2004</a>). First, to define the base line, the signal intensities of a pre-photobleach CFP (donor) and YFP (acceptor) were acquired by exciting with the 458 and 514-nm laser lines, respectively. Then, three regions of interest in the cell were selected: #1, Autofluorescence control; #2, non-photobleaching control; and #3, FRET-AB region. The CFP donor was excited with the 458 nm laser line for all FRET experiments; the emission of both CFP and YFP was recorded at 485 nm and 540 nm. Regions #1 and #3 were rendered free of YFP by consecutive cycles of bleaching recovery with the 514-nm laser line until no recovery of YFP was detected. The CFP and YFP signals were monitored throughout the experiment. After correction for background with control region #1 and for photobleaching of the donor because of imaging with control region #2, the FRET efficiencies (E) in the region #3 was calculated from the CFP signal using <span class="inline-formula" title="inline-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
FRET</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Eff</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
pre</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
post</div> </div> </div> </div> </div> </div> </div> </span>, where D is the mean intensity of the donor CFP in the area where the acceptor was bleached, before (Dpre) and after (Dpost) acceptor bleaching. The FRET efficiency is considered positive when Dpost &amp;gt; Dpre. The image and statistical analyses were performed with the FRET module for the ZEN software (Zeiss). The average FRET efficiency and its standard deviation were calculated from the FRET efficiencies of each individual cell in 27–30 cells per experiment. The standard Student's <i>t</i>-test was used to determine the statistical significance of the results. </p> </div> <div class="hmtasereconstitutionassay" title="sec"> <div class="title" tagx="title" title="title">
HMTase reconstitution assay</div> <p>In vitro HMTase reactions were modified from ( <a href="#bib43">Rea et al., 2000</a>; <a href="#bib52">Tachibana et al., 2001</a>) as follows: 20 µl of reaction mixture containing 3.3 µM Histone 3.2 (NEB), 1 µM His-GST-KYP, and 50 nCi of S-adenosyl-[methyl- <sup>14</sup>C]-L-methionine in HMTase buffer (50 mM Tris-HCl pH 9, 10 mM MgCl <sub>2</sub>, 1 mM ß-mercaptoethanol, 250 mM sucrose) was incubated for 0–10 min at 37°C. The reaction products were separated by 18% SDS-polyacrylamide gel electrophoresis and visualized by Coomassie Brilliant Blue R-250 staining; then, the gels were fixed 1 hr in fixing solution (25% Ethanol, 2% Glycerol) and scintillated for 30 min in 1 M sodium salicylate. Gels were dried 2 hr at 80°C. The <sup>14</sup>C signal was detected by fluorography using in a preflashed Classic autoradiography film blue sensitive; Filters Kodak Wratten No. 22 and No. 96 were used together for preflashing. Preflashed film was exposed 5–7 days at −80°C. The film was developed using a Kodak M35A X-OMAT Processor, and the results were digitalized in a Chemi-Doc XRS System and analyzed with the Image Lab Software (Bio-Rad, Hercules, CA, United States). </p> <p>To assess the impact of TrAP on KYP activity, His-MBP-TrAP or His-MBP were pre-incubated with His-GST-KYP in different molar ratios, ranging from 0 to 10, for 1 hr at room temperature, then the assays were proceeded as described above. The experiments were performed 3–5 times for statistical analysis. </p> </div> <div class="chipassays" title="sec"> <div class="title" tagx="title" title="title">
ChIP assays</div> <p>The analysis of histone modifications was performed as described ( <a href="#bib44a">Saleh et al., 2008</a>). Two grams of 9-day-old seedlings were crosslinked with 1% formaldehyde for 10 min by vacuum infiltration at 4°C; the reaction was stopped with 2 M Glycine to a final concentration of 100 mM at room temperature. Plants were rinsed 5 times with ice cold with water, flash-frozen in liquid nitrogen, and ground with mortar and pestle. The powder was suspended in 6 vol (12 ml) of nuclei isolation buffer (15 mM PIPES-KOH pH 6.8, 0.25 M sucrose, 0.9% Triton X-200, 5 mM MgCl <sub>2</sub>, 60 mM KCl, 15 mM NaCl, 1 mM CaCl <sub>2</sub>, 1 mM PMSF, 1 pellet/50 ml Complete EDTA-free Protease inhibitor [Roche]), filtered through two layers of Miracloth and centrifuged at 11,000 × rcf for 10 min in 4°C. After discarding the supernatant, the pellet was resuspended in 1 ml of Nuclei lysis buffer (50 mM HEPES pH 7.5, 1 mM EDTA pH 8.0, 150 mM NaCl, 1% SDS, 0.1% Sodium Deoxycholate, 1% Triton X-100, 1 pellet/50 ml Complete EDTA-free Protease inhibitor [Roche]); the samples were sonicated in 10 cycles 30 s ON and 90 s OFF, using the Bioruptor (Diagenode, Belgium) at the highest power in 4°Cs. The sonicated samples were centrifuged for 10 min at 21,000 × rcf in 4°C. 100 μl of the clarified chromatin was diluted 10-fold with Nuclei lysis buffer without SDS for each assay. The immunoprecipitation was accomplished by the addition of 40 µl Protein A Dynabeads (Invitrogen) and 3 μl of the pertinent antibody, followed by 6 hr incubation at 4°C on mild rotation. The beads-conjugated complexes were washed with 1 ml of Low salt buffer (20 mM Tris-HCl pH 8.0, 2 mM EDTA pH 8.0, 150 mM NaCl, 0.5% Triton X-100, 0.2% SDS), followed by 1 ml of high salt buffer (20 mM Tris-HCl pH 8.0, 2 mM EDTA pH 8.0, 500 mM NaCl, 0.5% Triton X-100, 0.2% SDS), then with 1 ml of LiCl buffer (10 mM Tris-HCl pH 8.0, 1 mM EDTA pH 8.0, 250 mM LiCl, 1% sodium deoxycholate, 1% NP-40), and finally twice with 1 ml of TE (10 mM Tris-HCl, 1 mM EDTA, pH 8.0) by incubating 5 min at 4°C in between washes. The samples were eluted twice at room temperature with 250 μl of elution buffer (100 mM NaHCO <sub>3</sub>, 0.5% SDS), for 15 and 30 min, respectively. The samples were decrosslinked with 100 mM NaCl at 65°C overnight, followed by Proteinase K treatment for 90 min at 45°C. The DNA was purified by Phenol:Chloroform:IsoamylAlcohol 25:24:1, and precipitated in 100% Ethanol at −80°C. </p> <p>The antibodies used are mono-clonal anti-H3K9me2 (Abcam, #Ab1220); monoclonal anti-H3K4me3 (Millipore, cat #04-745); mono-clonal anti-H3K27me3 (Millipore, cat #07-449). </p> <p>The immunoprecipitation of Flag-4Myc-KYP-Chromatin complexes was done as in ( <a href="#bib56a">Wierzbickiet al., 2008</a>), using Anti-FLAG M2 magnetic beads (Sigma-Aldrich, Cat# M8823). Two grams of rosette leaves 1–12 of mock or CaLCuV inoculated plants at 18 dpi were crosslinked with 1% formaldehyde for 25 min by vacuum infiltration at 4°C; the reaction was stopped with 2 M Glycine to a final concentration of 100 mM. Plants were rinsed five times with ice cold with water, flash-frozen in liquid nitrogen, and ground with mortar and pestle. The powder was suspended in 5 vol (10 ml) of Honda Buffer (20 mM HEPES-KOH pH 7.4, 0.44 M sucrose, 1.25% ficoll, 2.5% Dextran T40, 10 mM MgCl <sub>2</sub>, 0.5% Triton X-100, 5 mM DTT, 2 mM PMSF, 1 pellet/25 ml Complete EDTA-free Protease inhibitor [Roche]), filtered through two layers of Miracloth and centrifuged at 2000 × rcf for 15 min in 4°C. After discarding the supernatant, the nuclear precipitates were washed three times with 1 ml of Honda buffer; subsequently, the pellet was suspended in 300 μl of Nuclei lysis buffer (50 mM Tris-HCl pH 8.0, 10 mM EDTA pH 8.0, 1% SDS, 2 mM PMSF, 1 pellet/25 ml Complete EDTA-free Protease inhibitor [Roche]) and sonicated in ten cycles 30 s ON and 90 s OFF, using the Bioruptor (Diagenode) at the highest power in 4°Cs. The sonicated samples were centrifuged for 10 min at 21000 × rcf in 4°C. 100 μl of the clarified chromatin was diluted 10-fold with ChIP dilution buffer (16.7 mM Tris-HCl pH 8.0, 1.2 mM EDTA pH 8.0, 167 mM NaCl, 1.1% Triton X-100, 1 pellet/25 ml Complete EDTA-free Protease inhibitor [Roche]) per ChIP. The immunoprecipitation was accomplished by the addition of 40 µl of Anti-FLAG M2 magnetic beads (Sigma-Aldrich, Cat# M8823), followed by 2 hr incubation at 4°C on mild rotation. The beads were washed five times with 1 ml of Washing buffer (20 mM Tris-HCl pH 8.0, 2 mM EDTA pH 8.0, 150 mM NaCl, 1% Triton X-100, 1% SDS, 2 mM PMSF, 1 pellet/25 ml Complete EDTA-free Protease inhibitor [Roche]) incubating 5 min at 4°C in between; then, two more washes with 1 ml TE buffer incubating 5 min at 4°C. Finally, the samples were eluted twice at room temperature with 125 μl of Elution buffer (100 mM NaHCO <sub>3</sub>, 0.5% SDS), for 15 and 30 min, respectively. The samples were decrosslinked and the DNA extracted as above. </p> </div> <div class="microarrayanalysis" title="sec"> <div class="title" tagx="title" title="title">
Microarray analysis</div> <p>Microarray analyses using the Affymetrix ATH1 platform were performed with two biological replicates using wild-type plants, <i>35S-TrAP</i> transgenic plants, and <i>lhp1</i> mutants. Seedlings were grown on MS medium with 1% sucrose for 7 days. One mg of total RNA was used for reverse transcription using MessageAmp II aRNA kits (Ambion) and 15 mg of labeled cRNA for hybridization. GeneChip hybridization and scanning were performed at the Genomics Resource Center, Rockefeller University, New York. </p> <p>Statistical analysis of microarray data was performed using R software. Initially the microarray plates were tested for quality by an M plot and the data normalized by the RMA method from the Affy package. Subsequently, the distribution of the samples was assessed with scatter plots and the normalized data sets were evaluated with the Moderate t-test from R package limma for p-value computation. Then, the eBays method was used to compute moderated t-statistics and log-odds of differential expression by empirical Bayes shrinkage of the standard errors towards a common value. The moderated t-statistic (t) is the logFC to its standard error. In our DEG results our thresholds are p-value &amp;lt; 0.05 and logFC &amp;gt;1 (up-regulated) or logFC &amp;lt; −1 (down-regulated). The False Discovery Rate was approximated from the eBayes adjusted p-value. </p> <p>The significance of the overlapping data sets was calculated through Pearson's Chi-squared test with 1° of freedom. </p> </div> <div class="quantitativepcrandrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
Quantitative PCR and RT-PCR</div> <p>Expression levels of the tested genes were examined by quantitative RT-PCR. Total RNAs were prepared from 9 days-old seedlings and treated with DNase before being subjected to cDNA synthesis using Superscript III reverse transcriptase (Invitrogen) primed by random primers. The EF1a gene ( <a href="#bib57">Williams et al., 2005</a>) was included as an internal control for normalization. The enrichment levels of specific genes after ChIP assay were also tested by quantitative PCR. Primers are listed in <a href="#SD8-data">Supplementary file 8</a>. The quantitative PCRs were performed in 384-well plates with an ABI7900HT real-time PCR system using the SYBR Green I master mix (Applied Biosystems, Waltham, MA, United States) in a volume of 10 μl. PCR conditions were as follows: 50°C for 2 min, 95°C for 10 min, 45 cycles of 96°C for 10 s followed by 60°C for 1 min. Three biological repeats were performed, and the reactions were performed in triplicate for each run. The comparative CT method was used to evaluate the relative quantities of each amplified product in the samples. The threshold cycle (CT) was automatically determined for each reaction by the system according to the default parameters. The specificity of the PCR was determined by dissociation curve analysis of the amplified products using the standard method installed in the system. </p> </div> <div class="wholegenomebisulfitesequencing" title="sec"> <div class="title" tagx="title" title="title">
Whole genome bisulfite sequencing</div> <p>Approximately 500 ng of genomic DNA were used to generate libraries as described ( <a href="#bib21">Feng et al., 2011</a>) using premethylated adapters (NEXTFlex Bisulfite-Seq Adapters #511911, Bioo Scientific, Austin, TX, United States). The adaptor-ligated fragments were purified by QIAQuick column (Qiagen) and bisulfite converted using the EpiTect Kit (Qiagen, Germany) following the manufacturer's instructions. The converted DNA was later enriched by 15 cycles of PCR using the Pfu Turbo Cx Polymerase (Agilent, Santa Clara, CA, United States), using the specific primers provided by Bios Scientific for enrichment. The library was finally purified with Agencourt AMPure XP beads (Beckman Coulter, Pasadena, CA, United States) according to manufacturer's instruction. The libraries were single-end sequenced using HiSeq High Output with read length of 50 bp. Base calling and sequence cleaning was performed with the standard Illumina software, then the clean reads were mapped to the <i>Arabidopsis</i> genome (Version: TAIR10) using Bismark v0.14.3 ( <a href="#bib29">Krueger and Andrews, 2011</a>) with default parameters; the PCR duplicates were removed, and the methylation information was obtained with bismark with cutoff 3. The DMRs were obtained using swDMR ( <a href="#bib55">Wang et al., 2015</a>) with window 200, step size 100, (left 1000, right 1000), the samples were compared using the Kruskal–Wallis analysis of variance with p-value &amp;lt; 0.01. The DMRs were then annotated using BEDtools ( <a href="#bib41">Quinlan et al., 2010</a>). </p> </div> <div class="calcuvpathogenesisassays" title="sec"> <div class="title" tagx="title" title="title">
CaLCuV pathogenesis assays</div> <p>3-week-old Col-0 wild-type, <i>kyp</i> mutant and <i>Flag-4Myc-KYP</i> complemented were infected by agroinfiltration of the CaLCuV infective clones of DNAA and DNAB pNSB1090 and pNSB1091 ( <a href="#bib2">Arguello-Astorga et al., 2007</a>); the progression of the disease was evaluated daily in terms of time of symptom development and the severity of the symptoms observed. The assays were replicated 3 times, 36 plants of each genotype were used per assay, grown in short day conditions (8 hr light/16 hr dark). </p> <p>To assess systemic infection, we conducted virus infection assays using CaLCuV or CaLCuV Δ <i>trap</i> on thirty wild-type and thirty <i>kyp</i> mutant plants grown in short day condition (16 hr dark/8 hr light) at eight true leaves developmental stage, then we harvested nine to eleven newly emerged rosette leaves of each individual 18 days after inoculation ( <a href="#fig9">Figure 9B</a>). </p> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span> <span class="institution" title="institution">National Institutes of Health (NIH)</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">R21 AI097570</span> to Xiuren Zhang. <p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000928</span> <span class="institution" title="institution">Welch Foundation</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">A-1777</span> to Xiuren Zhang. <p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span> <span class="institution" title="institution">National Science Foundation (NSF)</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">MCB-1253369</span> to Xiuren Zhang. <p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Linda Hanley-Bowdoin, Judith Brown, and David Bisaro for the infective clones of the geminiviruses, Paul de Figueiredo and Hisashi Koiwa for imaging facilities, and Craig Kaplan for the sonication instrument. We thank Charlie Johnson, Richard Metz, and Sabrina Moisan at Texas A&amp;amp;M Genomics and Bioinformatics Service for their excellent support for the bisulfite sequencing. We thank Joshua Yuan and Xin Wang for advice and technical support for the proteomics analysis. We thank Robert Klein as well as members of Zhang's laboratory for stimulating discussions and critical review of the manuscript. The work was supported by grants from the US National Science Foundation (NSF CAREER) (MCB-1253369), the US National Institutes of Health (R21AI097570) and the Welch foundation (A-1777) to XZ. </p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"> <b>Competing interests</b> </div> <div class="fn-type-conflict" title="conflict"> <p>The authors declare that no competing interests exist.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"> <b>Author contributions</b> </div> <div class="fn-type-con" title="con"> <p>CC-G, Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>XZ, Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>XL, Conception and design, Acquisition of data, Analysis and interpretation of data.</p> </div> <div class="fn-type-con" title="con"> <p>X-JW, Conception and design, Acquisition of data, Analysis and interpretation of data.</p> </div> <div class="fn-type-con" title="con"> <p>CH, Acquisition of data, Analysis and interpretation of data, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>CZ, Acquisition of data, Analysis and interpretation of data, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>TH, Acquisition of data, Analysis and interpretation of data, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>FS, Acquisition of data, Analysis and interpretation of data, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>YZ, Acquisition of data, Analysis and interpretation of data, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>XZ, Acquisition of data, Analysis and interpretation of data, Contributed unpublished essential data or reagents. </p> </div> <div class="fn-type-con" title="con"> <p>ZM, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article. </p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.021">10.7554/eLife.06671.021</a> <span class="label" tagx="label" title="label">Supplementary file 1.</span>  <p>Microarray analysis of <i>35S-TrAP</i> transgenic plants. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.021">http://dx.doi.org/10.7554/eLife.06671.021</a> </p>  <div class="media" title="media"> <a href="elife06671s001.xlsx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.022">10.7554/eLife.06671.022</a> <span class="label" tagx="label" title="label">Supplementary file 2.</span>  <p>Transcriptome comparison of the genes deregulated by <i>TrAP</i> overexpression and <i>lhp1</i> loss-of-function mutant. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.022">http://dx.doi.org/10.7554/eLife.06671.022</a> </p>  <div class="media" title="media"> <a href="elife06671s002.xlsx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.023">10.7554/eLife.06671.023</a> <span class="label" tagx="label" title="label">Supplementary file 3.</span>  <p>Expression levels of genes encoding for TGS components in TrAP transgenic plants.</p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.023">http://dx.doi.org/10.7554/eLife.06671.023</a> </p>  <div class="media" title="media"> <a href="elife06671s003.xlsx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.024">10.7554/eLife.06671.024</a> <span class="label" tagx="label" title="label">Supplementary file 4.</span>  <p>List of proteins tested for interaction with TrAP.</p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.024">http://dx.doi.org/10.7554/eLife.06671.024</a> </p>  <div class="media" title="media"> <a href="elife06671s004.xlsx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.025">10.7554/eLife.06671.025</a> <span class="label" tagx="label" title="label">Supplementary file 5.</span>  <p>CG methylation analysis of Col-0 wild type, <i>kyp</i> mutant, and TrAP transgenic plants. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.025">http://dx.doi.org/10.7554/eLife.06671.025</a> </p>  <div class="media" title="media"> <a href="elife06671s005.xlsx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.026">10.7554/eLife.06671.026</a> <span class="label" tagx="label" title="label">Supplementary file 6.</span>  <p>CHG methylation analysis of Col-0 wild type, <i>kyp</i> mutant, and TrAP transgenic plants. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.026">http://dx.doi.org/10.7554/eLife.06671.026</a> </p>  <div class="media" title="media"> <a href="elife06671s006.xlsx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.027">10.7554/eLife.06671.027</a> <span class="label" tagx="label" title="label">Supplementary file 7.</span>  <p>CHH methylation analysis of Col-0 wild type, <i>kyp</i> mutant, and TrAP transgenic plants. </p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.027">http://dx.doi.org/10.7554/eLife.06671.027</a> </p>  <div class="media" title="media"> <a href="elife06671s007.xlsx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <a href="https://dx.doi.org/10.7554/eLife.06671.028">10.7554/eLife.06671.028</a> <span class="label" tagx="label" title="label">Supplementary file 8.</span>  <p>List of primers used in this article.</p> <p> <b>DOI:</b> <a href="10.7554/eLife.06671.028">http://dx.doi.org/10.7554/eLife.06671.028</a> </p>  <div class="media" title="media"> <a href="elife06671s008.xlsx">LINK</a> </div> </div> <div class="datasets" title="datasets"> <div class="title" tagx="title" title="title">
Major datasets</div> <p>The following datasets were generated:</p> <p> <span class="related-object" tagx="related-object" title="related-object"> <span class="collab'">collab: Castillo-González C</span>, <span class="collab'">collab: Zhang X</span>, <span class="collab'">collab: Liu X</span>, <span class="collab'">collab: Wang X</span>, <span class="year" tagx="year" title="year">2015</span> </span></p><div class="x" title="x">
, </div> <span class="source" tagx="source" title="source">TrAP expression in Arabidopsis thaliana lead to global reprogramming of endogenous genes </span> <div class="x" title="x">
, </div> <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73495">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73495</a> <div class="x" title="x">
, </div> <span class="comment">Publicly available at the NCBI Gene Expression Omnibus (Accession no: GSE73495)</span>.  <p /> <p> <span class="related-object" tagx="related-object" title="related-object"> <span class="collab'">collab: Castillo-González C</span>, <span class="collab'">collab: Zhang X</span>, <span class="collab'">collab: Liu X</span>, <span class="collab'">collab: Wang X</span>, <span class="year" tagx="year" title="year">2015</span> </span></p><div class="x" title="x">
, </div> <span class="source" tagx="source" title="source">Geminivirus-encoded TrAP suppressor inhibits the histone methyltransferase SUVH4/KYP to counter host defense </span> <div class="x" title="x">
, </div> <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73586">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73586</a> <div class="x" title="x">
, </div> <span class="comment">Publicly available at the NCBI Gene Expression Omnibus (Accession no:GSE73586)</span>.  <p /> <p>The following previously published dataset was used:</p> <p> <span class="related-object" tagx="related-object" title="related-object"> <span class="collab'">collab: Du J</span>, <span class="collab'">collab: Johnson LM</span>, <span class="collab'">collab: Groth M</span>, <span class="collab'">collab: Feng S</span>, <span class="collab'">collab: Hale CJ</span>, <span class="collab'">collab: Li S</span>, <span class="collab'">collab: Vashisht AA</span>, <span class="collab'">collab: Gallego-Bartolome J</span>, <span class="collab'">collab: Wohlschlegel JA</span>, <span class="collab'">collab: Patel DJ</span>, <span class="collab'">collab: Jacobsen SE</span>, <span class="year" tagx="year" title="year">2014</span> </span></p><div class="x" title="x">
, </div> <span class="source" tagx="source" title="source">Crystal structure of KRYPTONITE in complex with mCHH DNA and SAH</span> <div class="x" title="x">
, </div> <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4QEN">http://www.rcsb.org/pdb/explore/explore.do?structureId=4QEN</a> <div class="x" title="x">
, </div> <span class="comment">Publicly available at RCSB Protein Data Bank (Accession no. 4QEN).</span>  <p /> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="bib1" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aregger</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borah</span> <span class="given-names" tagx="given-names" title="given-names">BK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seguin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajeswaran</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gubaeva</span> <span class="given-names" tagx="given-names" title="given-names">EG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zvereva</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Windels</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vazquez</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blevins</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farinelli</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pooggin</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Primary and secondary siRNAs in geminivirus-induced gene silencing</span> <span class="source" tagx="source" title="source">PLOS Pathogens</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">9</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.ppat.1002941">10.1371/journal.ppat.1002941</a> </span> </span> </li> <li tag="ref"> <a name="bib2" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arguello-Astorga</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ascencio-Ibáñez</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dallas</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orozco</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanley-Bowdoin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">High-frequency reversion of geminivirus replication protein mutants during infection</span> <span class="source" tagx="source" title="source">Journal of Virology</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="fpage" tagx="fpage" title="fpage">11005</span> <span class="lpage" tagx="lpage" title="lpage">11015</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00925-07">10.1128/JVI.00925-07</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17670823">17670823</a> </span> </span> </li> <li tag="ref"> <a name="bib3" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baumberger</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baulcombe</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> ARGONAUTE1 is an RNA slicer that selectively recruits microRNAs and short interfering RNAs </span> <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of USA</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">11928</span> <span class="lpage" tagx="lpage" title="lpage">11933</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0505461102">10.1073/pnas.0505461102</a> </span> </span> </li> <li tag="ref"> <a name="bib4" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bisaro</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Silencing suppression by geminivirus proteins</span> <span class="source" tagx="source" title="source">Virology</span> <span class="volume" tagx="volume" title="volume">344</span> <span class="fpage" tagx="fpage" title="fpage">158</span> <span class="lpage" tagx="lpage" title="lpage">168</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.virol.2005.09.041">10.1016/j.virol.2005.09.041</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16364747">16364747</a> </span> </span> </li> <li tag="ref"> <a name="bib5" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Black</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Rechem</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whetstine</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Histone lysine methylation dynamics: establishment, regulation, and biological impact</span> <span class="source" tagx="source" title="source">Molecular Cell</span> <span class="volume" tagx="volume" title="volume">48</span> <span class="fpage" tagx="fpage" title="fpage">491</span> <span class="lpage" tagx="lpage" title="lpage">507</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molcel.2012.11.006">10.1016/j.molcel.2012.11.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23200123">23200123</a> </span> </span> </li> <li tag="ref"> <a name="bib6" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchmann</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asad</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">JN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohannath</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bisaro</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Geminivirus AL2 and L2 proteins suppress transcriptional gene silencing and cause genome-wide reductions in cytosine methylation </span> <span class="source" tagx="source" title="source">Journal of Virology</span> <span class="volume" tagx="volume" title="volume">83</span> <span class="fpage" tagx="fpage" title="fpage">5005</span> <span class="lpage" tagx="lpage" title="lpage">5013</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.01771-08">10.1128/JVI.01771-08</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19279102">19279102</a> </span> </span> </li> <li tag="ref"> <a name="bib7" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chanvivattana</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bishopp</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schubert</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stock</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moon</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sung</span> <span class="given-names" tagx="given-names" title="given-names">ZR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodrich</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Interaction of polycomb-group proteins controlling flowering in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">131</span> <span class="fpage" tagx="fpage" title="fpage">5263</span> <span class="lpage" tagx="lpage" title="lpage">5276</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1242/dev.01400">10.1242/dev.01400</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15456723">15456723</a> </span> </span> </li> <li tag="ref"> <a name="bib8" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chung</span> <span class="given-names" tagx="given-names" title="given-names">HY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lacatus</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunter</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Geminivirus AL2 protein induces expression of, and interacts with, a calmodulin-like gene, an endogenous regulator of gene silencing </span> <span class="source" tagx="source" title="source">Virology</span> <span class="volume" tagx="volume" title="volume">460–461</span> <span class="fpage" tagx="fpage" title="fpage">108</span> <span class="lpage" tagx="lpage" title="lpage">118</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.virol.2014.04.034">10.1016/j.virol.2014.04.034</a> </span> </span> </li> <li tag="ref"> <a name="bib9" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clough</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bent</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Floral dip: a simplified method for Agrobacterium-mediated transformation of <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Plant Journal</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">735</span> <span class="lpage" tagx="lpage" title="lpage">743</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-313x.1998.00343.x">10.1046/j.1365-313x.1998.00343.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10069079">10069079</a> </span> </span> </li> <li tag="ref"> <a name="bib10" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daelemans</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Costes</span> <span class="given-names" tagx="given-names" title="given-names">SV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">EH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Erwin-Cohen</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lockett</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pavlakis</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">In vivo HIV-1 rev multimerization in the nucleolus and cytoplasm identified by fluorescence resonance energy transfer </span> <span class="source" tagx="source" title="source">Journal of Biological Chemistry</span> <span class="volume" tagx="volume" title="volume">279</span> <span class="fpage" tagx="fpage" title="fpage">50167</span> <span class="lpage" tagx="lpage" title="lpage">50175</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M407713200">10.1074/jbc.M407713200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15294891">15294891</a> </span> </span> </li> <li tag="ref"> <a name="bib11" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deal</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henikoff</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Histone variants and modifications in plant gene regulation</span> <span class="source" tagx="source" title="source">Current Opinion in Plant Biology</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">116</span> <span class="lpage" tagx="lpage" title="lpage">122</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pbi.2010.11.005">10.1016/j.pbi.2010.11.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21159547">21159547</a> </span> </span> </li> <li tag="ref"> <a name="bib12" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voinnet</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Antiviral immunity directed by small RNAs</span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">413</span> <span class="lpage" tagx="lpage" title="lpage">426</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2007.07.039">10.1016/j.cell.2007.07.039</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17693253">17693253</a> </span> </span> </li> <li tag="ref"> <a name="bib13" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">du Chene</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Basyuk</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Triboulet</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knezevich</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chable-Bessia</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mettling</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baillat</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reynes</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corbeau</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bertrand</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marcello</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emiliani</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kiernan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benkirane</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Suv39H1 and HP1gamma are responsible for chromatin-mediated HIV-1 transcriptional silencing and post-integration latency </span> <span class="source" tagx="source" title="source">The EMBO Journal</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">424</span> <span class="lpage" tagx="lpage" title="lpage">435</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/sj.emboj.7601517">10.1038/sj.emboj.7601517</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17245432">17245432</a> </span> </span> </li> <li tag="ref"> <a name="bib14" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Groth</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hale</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vashisht</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego-Bartolome</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wohlschlegel</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2014a</span> <span class="mixed-article-title" title="mixed-article-title">Mechanism of DNA methylation-directed histone methylation by KRYPTONITE</span> <span class="source" tagx="source" title="source">Molecular Cell</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">495</span> <span class="lpage" tagx="lpage" title="lpage">504</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molcel.2014.06.009">10.1016/j.molcel.2014.06.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25018018">25018018</a> </span> </span> </li> <li tag="ref"> <a name="bib15" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Groth</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hale</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vashisht</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego-Bartolome</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wohlschlegel</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2014b</span> <div class="unknown" title="data-title" tagxxx="data-title">
Crystal structure of KRYPTONITE in complex with mCHH DNA and SAH</div> <span class="source" tagx="source" title="source">RCSB Protein Data Bank</span> <span class="comment"> <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4QEN">http://www.rcsb.org/pdb/explore/explore.do?structureId=4QEN</a> </span> </span> </li> <li tag="ref"> <a name="bib16" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernatavichute</span> <span class="given-names" tagx="given-names" title="given-names">YV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stroud</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caro</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vashisht</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terragni</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chin</span> <span class="given-names" tagx="given-names" title="given-names">HG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tu</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hetzel</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wohlschlegel</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pradhan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Dual binding of chromomethylase domains to H3K9me2-containing nucleosomes directs DNA methylation in plants </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">151</span> <span class="fpage" tagx="fpage" title="fpage">167</span> <span class="lpage" tagx="lpage" title="lpage">180</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2012.07.034">10.1016/j.cell.2012.07.034</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23021223">23021223</a> </span> </span> </li> <li tag="ref"> <a name="bib17" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubin</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Remigereau</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osborne</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casale</span> <span class="given-names" tagx="given-names" title="given-names">FP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drewe</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kahles</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jean</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vilhjálmsson</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jagoda</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irez</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voronin</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Long</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rätsch</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stegle</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MClark</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nordborg</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">DNA methylation in <i>Arabidopsis</i> has a genetic basis and shows evidence of local adaptation </span> <span class="source" tagx="source" title="source">eLife</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">e05255</span> <span class="pub-id"> <a href="https://dx.doi.org/10.7554/eLife.05255">10.7554/eLife.05255</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25939354">25939354</a> </span> </span> </li> <li tag="ref"> <a name="bib18" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eschenfeldt</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lucy</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millard</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joachimiak</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mark</span> <span class="given-names" tagx="given-names" title="given-names">ID</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">A family of LIC vectors for high-throughput cloning and purification of proteins</span> <span class="source" tagx="source" title="source">Methods in Molecular Biology</span> <span class="volume" tagx="volume" title="volume">498</span> <span class="fpage" tagx="fpage" title="fpage">105</span> <span class="lpage" tagx="lpage" title="lpage">115</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-1-59745-196-3_7">10.1007/978-1-59745-196-3_7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18988021">18988021</a> </span> </span> </li> <li tag="ref"> <a name="bib19" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farrona</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coupland</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turck</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">The impact of chromatin regulation on the floral transition</span> <span class="source" tagx="source" title="source">Seminars in Cell &amp;amp; Developmental Biology</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">560</span> <span class="lpage" tagx="lpage" title="lpage">573</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.semcdb.2008.07.015">10.1016/j.semcdb.2008.07.015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18708152">18708152</a> </span> </span> </li> <li tag="ref"> <a name="bib20" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Epigenetic modifications in plants: an evolutionary perspective</span> <span class="source" tagx="source" title="source">Current Opinion in Plant Biology</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">179</span> <span class="lpage" tagx="lpage" title="lpage">186</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pbi.2010.12.002">10.1016/j.pbi.2010.12.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21233005">21233005</a> </span> </span> </li> <li tag="ref"> <a name="bib21" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubbi</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pellegrini</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Determining DNA methylation profiles using sequencing</span> <span class="source" tagx="source" title="source">Methods in Molecular Biology</span> <span class="volume" tagx="volume" title="volume">733</span> <span class="fpage" tagx="fpage" title="fpage">223</span> <span class="lpage" tagx="lpage" title="lpage">238</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-1-61779-089-8_16">10.1007/978-1-61779-089-8_16</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21431774">21431774</a> </span> </span> </li> <li tag="ref"> <a name="bib22" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gan</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ito</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Functional roles of histone modification, chromatin remodeling and microRNAs in <i>Arabidopsis</i> flower development </span> <span class="source" tagx="source" title="source">International Review of Cell and Molecular Biology</span> <span class="volume" tagx="volume" title="volume">305</span> <span class="fpage" tagx="fpage" title="fpage">115</span> <span class="lpage" tagx="lpage" title="lpage">161</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/B978-0-12-407695-2.00003-2">10.1016/B978-0-12-407695-2.00003-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23890381">23890381</a> </span> </span> </li> <li tag="ref"> <a name="bib23" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greer</span> <span class="given-names" tagx="given-names" title="given-names">EL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="source" tagx="source" title="source">Nature Reviews Genetics</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">343</span> <span class="lpage" tagx="lpage" title="lpage">357</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg3173">10.1038/nrg3173</a> </span> </span> </li> <li tag="ref"> <a name="bib24" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanley-Bowdoin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bejarano</span> <span class="given-names" tagx="given-names" title="given-names">ER</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robertson</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mansoor</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Geminiviruses: masters at redirecting and reprogramming plant processes</span> <span class="source" tagx="source" title="source">Nature Reviews Microbiology</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">777</span> <span class="lpage" tagx="lpage" title="lpage">788</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrmicro3117">10.1038/nrmicro3117</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24100361">24100361</a> </span> </span> </li> <li tag="ref"> <a name="bib25" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayes</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buck</span> <span class="given-names" tagx="given-names" title="given-names">KW</span> </span> </span> <span class="year" tagx="year" title="year">1989</span> <span class="mixed-article-title" title="mixed-article-title">Replication of tomato golden mosaic virus DNA B in transgenic plants expressing open reading frames (ORFs) of DNA A: requirement of ORF AL2 for production of single-stranded DNA </span> <span class="source" tagx="source" title="source">Nucleic Acids Research</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">10213</span> <span class="lpage" tagx="lpage" title="lpage">10222</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/17.24.10213">10.1093/nar/17.24.10213</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2602150">2602150</a> </span> </span> </li> <li tag="ref"> <a name="bib25a" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Højfeldt</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agger</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Helin</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Histone lysine demethylases as targets for anticancer therapy</span> <span class="source" tagx="source" title="source">Nature Reviews Drug Discovery</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">917</span> <span class="lpage" tagx="lpage" title="lpage">930</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrd4154">10.1038/nrd4154</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24232376">24232376</a> </span> </span> </li> <li tag="ref"> <a name="bib26" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kenworthy</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Imaging protein-protein interactions using fluorescence resonance energy transfer microscopy </span> <span class="source" tagx="source" title="source">Methods</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">289</span> <span class="lpage" tagx="lpage" title="lpage">296</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/meth.2001.1189">10.1006/meth.2001.1189</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11403577">11403577</a> </span> </span> </li> <li tag="ref"> <a name="bib27" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kotake</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takada</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakahigashi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohto</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> TERMINAL FLOWER 2 gene encodes a heterochromatin protein 1 homolog and represses both FLOWERING LOCUS T to regulate flowering time and several floral homeotic genes </span> <span class="source" tagx="source" title="source">Plant &amp;amp; Cell Physiology</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">555</span> <span class="lpage" tagx="lpage" title="lpage">564</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcg091">10.1093/pcp/pcg091</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12826620">12826620</a> </span> </span> </li> <li tag="ref"> <a name="bib28" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kouzarides</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Chromatin modifications and their function</span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">128</span> <span class="fpage" tagx="fpage" title="fpage">693</span> <span class="lpage" tagx="lpage" title="lpage">705</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2007.02.005">10.1016/j.cell.2007.02.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17320507">17320507</a> </span> </span> </li> <li tag="ref"> <a name="bib29" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krueger</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andrews</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">1571</span> <span class="lpage" tagx="lpage" title="lpage">1572</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btr167">10.1093/bioinformatics/btr167</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21493656">21493656</a> </span> </span> </li> <li tag="ref"> <a name="bib30" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lacatus</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunter</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis</i> PEAPOD2 transcription factor interacts with geminivirus AL2 protein and the coat protein promoter </span> <span class="source" tagx="source" title="source">Virology</span> <span class="volume" tagx="volume" title="volume">392</span> <span class="fpage" tagx="fpage" title="fpage">196</span> <span class="lpage" tagx="lpage" title="lpage">202</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.virol.2009.07.004">10.1016/j.virol.2009.07.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19647846">19647846</a> </span> </span> </li> <li tag="ref"> <a name="bib31" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">FL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">XF</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Histone methylation in higher plants</span> <span class="source" tagx="source" title="source">Annual Review of Plant Biology</span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">395</span> <span class="lpage" tagx="lpage" title="lpage">420</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.043008.091939">10.1146/annurev.arplant.043008.091939</a> </span> </span> </li> <li tag="ref"> <a name="bib32" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">ZW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">JX</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">HW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">XJ</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">The SET domain proteins SUVH2 and SUVH9 are required for Pol V occupancy at RNA-directed DNA methylation loci </span> <span class="source" tagx="source" title="source">PLOS Genetics</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">e1003948</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pgen.1003948">10.1371/journal.pgen.1003948</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24465213">24465213</a> </span> </span> </li> <li tag="ref"> <a name="bib33" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lozano-Durán</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosas-Díaz</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gusmaroli</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luna</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taconnat</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deng</span> <span class="given-names" tagx="given-names" title="given-names">XW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bejarano</span> <span class="given-names" tagx="given-names" title="given-names">ER</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Geminiviruses subvert Ubiquitination by altering CSN-mediated derubylation of SCF E3 ligase complexes and inhibit jasmonate signaling in <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">1014</span> <span class="lpage" tagx="lpage" title="lpage">1032</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.110.080267">10.1105/tpc.110.080267</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21441437">21441437</a> </span> </span> </li> <li tag="ref"> <a name="bib34" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moffatt</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stevens</span> <span class="given-names" tagx="given-names" title="given-names">YY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allen</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Snider</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pereira</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Todorova</span> <span class="given-names" tagx="given-names" title="given-names">MI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Summers</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weretilnyk</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin-McCaffrey</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Adenosine kinase deficiency is associated with developmental abnormalities and reduced transmethylation </span> <span class="source" tagx="source" title="source">Plant Physiology</span> <span class="volume" tagx="volume" title="volume">128</span> <span class="fpage" tagx="fpage" title="fpage">812</span> <span class="lpage" tagx="lpage" title="lpage">821</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.010880">10.1104/pp.010880</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11891238">11891238</a> </span> </span> </li> <li tag="ref"> <a name="bib35" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morel</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godon</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mourrain</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Béclin</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boutet</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feuerbach</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Proux</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaucheret</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Fertile hypomorphic ARGONAUTE (ago1) mutants impaired in post-transcriptional gene silencing and virus resistance </span> <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">629</span> <span class="lpage" tagx="lpage" title="lpage">639</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.010358">10.1105/tpc.010358</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11910010">11910010</a> </span> </span> </li> <li tag="ref"> <a name="bib36" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murashige</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skoog</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="year" tagx="year" title="year">1962</span> <span class="mixed-article-title" title="mixed-article-title">A revised medium for rapid growth and Bio assays with tobacco tissue cultures</span> <span class="source" tagx="source" title="source">Physiologia Plantarum</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">473</span> <span class="lpage" tagx="lpage" title="lpage">497</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1399-3054.1962.tb08052.x">10.1111/j.1399-3054.1962.tb08052.x</a> </span> </span> </li> <li tag="ref"> <a name="bib37" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakahigashi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jasencakova</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schubert</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis</i> heterochromatin protein1 homolog (TERMINAL FLOWER2) silences genes within the euchromatic region but not genes positioned in heterochromatin </span> <span class="source" tagx="source" title="source">Plant &amp;amp; Cell Physiology</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">1747</span> <span class="lpage" tagx="lpage" title="lpage">1756</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pci195">10.1093/pcp/pci195</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16131496">16131496</a> </span> </span> </li> <li tag="ref"> <a name="bib38" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narasipura</span> <span class="given-names" tagx="given-names" title="given-names">SD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Harthi</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Epigenetic regulation of HIV-1 latency in astrocytes</span> <span class="source" tagx="source" title="source">Journal of Virology</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">3031</span> <span class="lpage" tagx="lpage" title="lpage">3038</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.03333-13">10.1128/JVI.03333-13</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24352441">24352441</a> </span> </span> </li> <li tag="ref"> <a name="bib39" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ham</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">BS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roh</span> <span class="given-names" tagx="given-names" title="given-names">TY</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of histone modifications in latently HIV-1 infected T cells</span> <span class="source" tagx="source" title="source">AIDS</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">1719</span> <span class="lpage" tagx="lpage" title="lpage">1728</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1097/QAD.0000000000000309">10.1097/QAD.0000000000000309</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24762674">24762674</a> </span> </span> </li> <li tag="ref"> <a name="bib40" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pumplin</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voinnet</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">RNA silencing suppression by plant pathogens: defence, counter-defence and counter-counter-defence</span> <span class="source" tagx="source" title="source">Nature Reviews Microbiology</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">745</span> <span class="lpage" tagx="lpage" title="lpage">760</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrmicro3120">10.1038/nrmicro3120</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24129510">24129510</a> </span> </span> </li> <li tag="ref"> <a name="bib41" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quinlan</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">IM</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">841</span> <span class="lpage" tagx="lpage" title="lpage">842</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278">20110278</a> </span> </span> </li> <li tag="ref"> <a name="bib42" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raja</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanville</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchmann</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bisaro</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Viral genome methylation as an epigenetic defense against geminiviruses</span> <span class="source" tagx="source" title="source">Journal of Virology</span> <span class="volume" tagx="volume" title="volume">82</span> <span class="fpage" tagx="fpage" title="fpage">8997</span> <span class="lpage" tagx="lpage" title="lpage">9007</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00719-08">10.1128/JVI.00719-08</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18596098">18596098</a> </span> </span> </li> <li tag="ref"> <a name="bib43" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rea</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eisenhaber</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Carroll</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strahl</span> <span class="given-names" tagx="given-names" title="given-names">BD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">ZW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmid</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Opravil</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mechtler</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ponting</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allis</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenuwein</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">406</span> <span class="fpage" tagx="fpage" title="fpage">593</span> <span class="lpage" tagx="lpage" title="lpage">599</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/35020506">10.1038/35020506</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10949293">10949293</a> </span> </span> </li> <li tag="ref"> <a name="bib44" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez-Negrete</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lozano-Duran</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Piedra-Aguilera</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cruzado</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bejarano</span> <span class="given-names" tagx="given-names" title="given-names">ER</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castillo</span> <span class="given-names" tagx="given-names" title="given-names">AG</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Geminivirus Rep protein interferes with the plant DNA methylation machinery and suppresses transcriptional gene silencing </span> <span class="source" tagx="source" title="source">New Phytologist</span> <span class="volume" tagx="volume" title="volume">199</span> <span class="fpage" tagx="fpage" title="fpage">464</span> <span class="lpage" tagx="lpage" title="lpage">475</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/nph.12286">10.1111/nph.12286</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23614786">23614786</a> </span> </span> </li> <li tag="ref"> <a name="bib44a" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saleh</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alvarez-Venegas</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avramova</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">An efficient chromatin immunoprecipitation (ChIP) protocol for studying histone modifications in <i>Arabidopsis</i> plants </span> <span class="source" tagx="source" title="source">Nature Protocols</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">1018</span> <span class="lpage" tagx="lpage" title="lpage">1025</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2008.66">10.1038/nprot.2008.66</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18536649">18536649</a> </span> </span> </li> <li tag="ref"> <a name="bib45" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dallas</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goshe</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanley-Bowdoin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">SnRK1 phosphorylation of AL2 delays cabbage leaf curl virus infection in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">Journal of Virology</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">10598</span> <span class="lpage" tagx="lpage" title="lpage">10612</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00761-14">10.1128/JVI.00761-14</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24990996">24990996</a> </span> </span> </li> <li tag="ref"> <a name="bib46" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reyes</span> <span class="given-names" tagx="given-names" title="given-names">MI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanley-Bowdoin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> protein kinases GRIK1 and GRIK2 specifically activate SnRK1 by phosphorylating its activation loop </span> <span class="source" tagx="source" title="source">Plant Physiology</span> <span class="volume" tagx="volume" title="volume">150</span> <span class="fpage" tagx="fpage" title="fpage">996</span> <span class="lpage" tagx="lpage" title="lpage">1005</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.108.132787">10.1104/pp.108.132787</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19339507">19339507</a> </span> </span> </li> <li tag="ref"> <a name="bib47" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Jonge</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forsberg</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pettersson</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheng</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hennig</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carlborg</span> <span class="given-names" tagx="given-names" title="given-names">Ö</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Natural CMT2 variation is associated with genome-wide methylation changes and temperature seasonality </span> <span class="source" tagx="source" title="source">PLOS Genetics</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">e1004842</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pgen.1004842">10.1371/journal.pgen.1004842</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25503602">25503602</a> </span> </span> </li> <li tag="ref"> <a name="bib48" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shung</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunter</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Regulation of Tomato golden mosaic virus AL2 and AL3 gene expression by a conserved upstream open reading frame </span> <span class="source" tagx="source" title="source">Virology</span> <span class="volume" tagx="volume" title="volume">383</span> <span class="fpage" tagx="fpage" title="fpage">310</span> <span class="lpage" tagx="lpage" title="lpage">318</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.virol.2008.10.020">10.1016/j.virol.2008.10.020</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19012941">19012941</a> </span> </span> </li> <li tag="ref"> <a name="bib49" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunter</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bisaro</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="year" tagx="year" title="year">1992</span> <span class="mixed-article-title" title="mixed-article-title">Transactivation of geminivirus AR1 and BR1 gene expression by the viral AL2 gene product occurs at the level of transcription </span> <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">1321</span> <span class="lpage" tagx="lpage" title="lpage">1331</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.4.10.1321">10.1105/tpc.4.10.1321</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1446172">1446172</a> </span> </span> </li> <li tag="ref"> <a name="bib50" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stroud</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Do</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Non-CG methylation patterns shape the epigenetic landscape in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">Nature Structural &amp;amp; Molecular Biology</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">64</span> <span class="lpage" tagx="lpage" title="lpage">72</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nsmb.2735">10.1038/nsmb.2735</a> </span> </span> </li> <li tag="ref"> <a name="bib51" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stroud</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greenberg</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernatavichute</span> <span class="given-names" tagx="given-names" title="given-names">YV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Comprehensive analysis of silencing mutants reveals complex regulation of the <i>Arabidopsis</i> methylome </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">152</span> <span class="fpage" tagx="fpage" title="fpage">352</span> <span class="lpage" tagx="lpage" title="lpage">364</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2012.10.054">10.1016/j.cell.2012.10.054</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23313553">23313553</a> </span> </span> </li> <li tag="ref"> <a name="bib52" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tachibana</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugimoto</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukushima</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinkai</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Set domain-containing protein, G9a, is a novel lysine-preferring mammalian histone methyltransferase with hyperactivity and specific selectivity to lysines 9 and 27 of histone H3 </span> <span class="source" tagx="source" title="source">The Journal of Biological Chemistry</span> <span class="volume" tagx="volume" title="volume">276</span> <span class="fpage" tagx="fpage" title="fpage">25309</span> <span class="lpage" tagx="lpage" title="lpage">25317</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M101914200">10.1074/jbc.M101914200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11316813">11316813</a> </span> </span> </li> <li tag="ref"> <a name="bib53" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turck</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roudier</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farrona</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin-Magniette</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guillaume</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buisine</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gagnot</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martienssen</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coupland</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colot</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> TFL2/LHP1 specifically associates with genes marked by trimethylation of histone H3 lysine 27 </span> <span class="source" tagx="source" title="source">PLOS Genetics</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">e86</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pgen.0030086">10.1371/journal.pgen.0030086</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17542647">17542647</a> </span> </span> </li> <li tag="ref"> <a name="bib54" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buckley</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchmann</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bisaro</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Adenosine kinase inhibition and suppression of RNA silencing by geminivirus AL2 and L2 proteins </span> <span class="source" tagx="source" title="source">Journal of Virology</span> <span class="volume" tagx="volume" title="volume">79</span> <span class="fpage" tagx="fpage" title="fpage">7410</span> <span class="lpage" tagx="lpage" title="lpage">7418</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.79.12.7410-7418.2005">10.1128/JVI.79.12.7410-7418.2005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15919897">15919897</a> </span> </span> </li> <li tag="ref"> <a name="bib55" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">swDMR: a sliding window approach to identify differentially methylated regions based on whole genome bisulfite sequencing </span> <span class="source" tagx="source" title="source">PLOS ONE</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">e0132866</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0132866">10.1371/journal.pone.0132866</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26176536">26176536</a> </span> </span> </li> <li tag="ref"> <a name="bib56" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weretilnyk</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alexander</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drebenstedt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Snider</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Summers</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moffatt</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Maintaining methylation activities during salt stress. The involvement of adenosine kinase </span> <span class="source" tagx="source" title="source">Plant Physiology</span> <span class="volume" tagx="volume" title="volume">125</span> <span class="fpage" tagx="fpage" title="fpage">856</span> <span class="lpage" tagx="lpage" title="lpage">865</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.125.2.856">10.1104/pp.125.2.856</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11161043">11161043</a> </span> </span> </li> <li tag="ref"> <a name="bib56a" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wierzbicki</span> <span class="given-names" tagx="given-names" title="given-names">AT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haag</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pikaard</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Noncoding transcription by RNA polymerase Pol IVb/Pol V mediates transcriptional silencing of overlapping and adjacent genes </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">135</span> <span class="fpage" tagx="fpage" title="fpage">635</span> <span class="lpage" tagx="lpage" title="lpage">648</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2008.09.035">10.1016/j.cell.2008.09.035</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19013275">19013275</a> </span> </span> </li> <li tag="ref"> <a name="bib57" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grigg</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christensen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fletcher</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Regulation of <i>Arabidopsis</i> shoot apical meristem and lateral organ formation by microRNA miR166g and its AtHD-ZIP target genes </span> <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">132</span> <span class="fpage" tagx="fpage" title="fpage">3657</span> <span class="lpage" tagx="lpage" title="lpage">3668</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1242/dev.01942">10.1242/dev.01942</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16033795">16033795</a> </span> </span> </li> <li tag="ref"> <a name="bib58" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">You</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Whole-genome DNA methylation patterns and complex associations with gene structure and expression during flower development in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">The Plant Journal</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="fpage" tagx="fpage" title="fpage">268</span> <span class="lpage" tagx="lpage" title="lpage">281</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/tpj.12726">10.1111/tpj.12726</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25404462">25404462</a> </span> </span> </li> <li tag="ref"> <a name="bib59" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baliji</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchmann</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindbo</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunter</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bisaro</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Functional modulation of the geminivirus AL2 transcription factor and silencing suppressor by self-interaction </span> <span class="source" tagx="source" title="source">Journal of Virology</span> <span class="volume" tagx="volume" title="volume">81</span> <span class="fpage" tagx="fpage" title="fpage">11972</span> <span class="lpage" tagx="lpage" title="lpage">11981</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JVI.00617-07">10.1128/JVI.00617-07</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17715241">17715241</a> </span> </span> </li> <li tag="ref"> <a name="bib60" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clarenz</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cokus</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernatavichute</span> <span class="given-names" tagx="given-names" title="given-names">YV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pellegrini</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodrich</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2007a</span> <span class="mixed-article-title" title="mixed-article-title">Whole-genome analysis of histone H3 lysine 27 trimethylation in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">PLOS Biology</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">e129</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pbio.0050129">10.1371/journal.pbio.0050129</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17439305">17439305</a> </span> </span> </li> <li tag="ref"> <a name="bib61" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garreton</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">The AIP2 E3 ligase acts as a novel negative regulator of ABA signaling by promoting ABI3 degradation </span> <span class="source" tagx="source" title="source">Genes &amp;amp; Development</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1532</span> <span class="lpage" tagx="lpage" title="lpage">1543</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.1318705">10.1101/gad.1318705</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15998807">15998807</a> </span> </span> </li> <li tag="ref"> <a name="bib62" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Germann</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blus</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khorasanizadeh</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaudin</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> </span> <span class="year" tagx="year" title="year">2007b</span> <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis</i> LHP1 protein colocalizes with histone H3 Lys27 trimethylation </span> <span class="source" tagx="source" title="source">Nature Structural &amp;amp; Molecular Biology</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">869</span> <span class="lpage" tagx="lpage" title="lpage">871</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nsmb1283">10.1038/nsmb1283</a> </span> </span> </li> <li tag="ref"> <a name="bib63" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henriques</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niu</span> <span class="given-names" tagx="given-names" title="given-names">QW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span> </span> <span class="year" tagx="year" title="year">2006a</span> <span class="mixed-article-title" title="mixed-article-title">Agrobacterium-mediated transformation of <i>Arabidopsis thaliana</i> using the floral-dip method </span> <span class="source" tagx="source" title="source">Nature Protocols</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">641</span> <span class="lpage" tagx="lpage" title="lpage">646</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2006.97">10.1038/nprot.2006.97</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17406292">17406292</a> </span> </span> </li> <li tag="ref"> <a name="bib64" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">YR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pei</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuschl</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span> </span> <span class="year" tagx="year" title="year">2006b</span> <span class="mixed-article-title" title="mixed-article-title">Cucumber mosaic virus-encoded 2b suppressor inhibits <i>Arabidopsis Argonaute</i>1 cleavage activity to counter plant defense </span> <span class="source" tagx="source" title="source">Genes &amp;amp; Development</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">3255</span> <span class="lpage" tagx="lpage" title="lpage">3268</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.1495506">10.1101/gad.1495506</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17158744">17158744</a> </span> </span> </li> <li tag="ref"> <a name="bib65" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lai</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teng</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">BSCTV C2 attenuates the degradation of SAMDC1 to suppress DNA methylation-mediated gene silencing in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">273</span> <span class="lpage" tagx="lpage" title="lpage">288</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.110.081695">10.1105/tpc.110.081695</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21245466">21245466</a> </span> </span> </li> <li tag="ref"> <a name="bib66" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Dynamics of histone H3 lysine 27 trimethylation in plant development</span> <span class="source" tagx="source" title="source">Current Opinion in Plant Biology</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">123</span> <span class="lpage" tagx="lpage" title="lpage">129</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pbi.2011.01.001">10.1016/j.pbi.2011.01.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21330185">21330185</a> </span> </span> </li> <li tag="ref"> <a name="bib67" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castillo-González</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">YT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axtell</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">XJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Bidirectional processing of pri-miRNAs with branched terminal loops by <i>Arabidopsis</i> Dicer-like1 </span> <span class="source" tagx="source" title="source">Nature Structural &amp;amp; Molecular Biology</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">1106</span> <span class="lpage" tagx="lpage" title="lpage">1115</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nsmb.2646">10.1038/nsmb.2646</a> </span> </span> </li> <li tag="ref"> <a name="bib68" /> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuo</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niu</span> <span class="given-names" tagx="given-names" title="given-names">QW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Technical advance: an estrogen receptor-based transactivator XVE mediates highly inducible gene expression in transgenic plants </span> <span class="source" tagx="source" title="source">Plant Journal</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">265</span> <span class="lpage" tagx="lpage" title="lpage">273</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-313x.2000.00868.x">10.1046/j.1365-313x.2000.00868.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11069700">11069700</a> </span> </span> </li> </ul> </div> </div>  </body></html>